<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Venkat Malladi" />
  <meta name="dcterms.date" content="2017-11-07" />
  <meta name="keywords" content="enhancers, transcription, transcription factor, epigenome" />
  <title>Exploring Lineage-Specific Enhancers by Integrating Enhancer Transcription, Epigenomic Features, Sequence Motifs, and Transcription Factor Expression</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      .smallcaps{font-variant: small-caps;}
      .line-block{white-space: pre-line;}
      .column{display: inline-block;}
  </style>
  <link rel="stylesheet" href="github-pandoc.css">
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_CHTML-full" type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!-- Insert Analytics Script Below -->
  <script>
  </script>
  <!-- End Analytics Script -->
</head>
<body>
<header>
<h1 class="title">Exploring Lineage-Specific Enhancers by Integrating Enhancer Transcription, Epigenomic Features, Sequence Motifs, and Transcription Factor Expression</h1>
</header>
<p><small><em> This manuscript was automatically generated from <a href="https://github.com/vsmalladi/tfsee-manuscript/tree/d873572a766c88335c1973f4d67d3edd7493ce1f">vsmalladi/tfsee-manuscript@d873572</a> on November 7, 2017. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><strong>Venkat Malladi</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-0144-0564">0000-0002-0144-0564</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/vsmalladi">vsmalladi</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/katatonikkat">katatonikkat</a><br> <small> </small></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>The identification of transcription factors (TF) driving the formation of active enhancers that regulate the expression of target genes remains an open problem. We have developed a computational framework that identifies cell type-specific enhancers and their cognate TFs by integrating multiple genomic assays that probe the transcriptomes (GRO-seq and RNA-seq) and epigenomes (ChIP-seq) of various samples. Our method, called Total Functional Score of Enhancer Elements (TFSEE), integrates the magnitude of enhancer transcription (GRO-seq), enrichment of marks associated with enhancers (H3K4me1 and H3K27ac ChIP-seq), TF mRNA expression levels (RNA-seq), and TF motif p-values (MEME). This method has allowed us to explore the enhancer landscape in different cell types that share common origins or are biologically related, including distinct molecular subtypes of breast cancer, and embryonic stem cells (ESCs) and their derived lineages. Using TFSEE, we have identified key breast cancer subtype-specific transcription factors that are bound at active enhancers and dictate gene expression patterns determining growth outcomes. To demonstrate the broader utility of our approach, we have used this algorithm to identify transcription factors during the differentiation of embryonic stem cells into pancreatic cells. Taken together our results show that TFSEE can be used to perform multilayer genomic data integration to uncover novel cell type-specific transcription factors that control lineage-specific enhancers.</p>
<h2 id="introduction">Introduction</h2>
<h2 id="results">Results</h2>
<h2 id="discussion">Discussion</h2>
<h2 id="acknowledgments">Acknowledgments</h2>
<h2 id="material-and-methods" class="page_break_before">Material and Methods</h2>
<h3 id="genomic-data-curation">Genomic Data Curation</h3>
<p>We used previously published GRO-seq, ChIP-seq and RNA-seq data from <span class="citation" data-cites="BTqz3DNL 18a6G1TE5">[<a href="#ref-BTqz3DNL">1</a>,<a href="#ref-18a6G1TE5">2</a>]</span> of time course differentiation of human embryonic stem cells (hESC) to pancreatic endoderm (PE). All data sets are available from NCBI’s Gene Expression Omnibus <span class="citation" data-cites="Bc3QGVe7">[<a href="#ref-Bc3QGVe7">3</a>]</span> or EMBL-EBI’s ArrayExpress <span class="citation" data-cites="937jwJMM">[<a href="#ref-937jwJMM">4</a>]</span> repositories using the accession numbers listed in Table <a href="#tbl:data-sets">1</a>.</p>
<a name="tbl:data-sets"></a>
<table style="width:21%;">
<caption>Table 1: <strong>Description and accession numbers of GRO-seq, ChIP-seq and RNA-seq datasets.</strong> </caption>
<colgroup>
<col style="width: 6%" />
<col style="width: 13%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Assay</th>
<th style="text-align: left;">Accessions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">GRO-seq</td>
<td style="text-align: left;">GSM1316306, GSM1316313, GSM1316320, GSM1316327, GSM1316334</td>
</tr>
<tr class="even">
<td style="text-align: left;">H3K4me3 ChIP-seq</td>
<td style="text-align: left;">ERR208008, ERR208014, ERR207998, ERR20798, ERR207999</td>
</tr>
<tr class="odd">
<td style="text-align: left;">H3K4me1 ChIP-seq</td>
<td style="text-align: left;">GSM1316302, GSM1316303, GSM1316309, GSM1316316, GSM1316317, GSM1316310, GSM1316323, GSM1316324, GSM1316330, GSM1316331</td>
</tr>
<tr class="even">
<td style="text-align: left;">H3K27ac ChIP-seq</td>
<td style="text-align: left;">GSM1316300, GSM1316301, GSM1316307, GSM1316308, GSM1316314, GSM1316315, GSM1316321, GSM1316322, GSM1316328, GSM1316329</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Input ChIP-seq</td>
<td style="text-align: left;">ERR208001, ERR208012, ERR207984, ERR208011, ERR207986, GSM1316304, GSM1316305, GSM1316311, GSM1316312, GSM1316318, GSM1316319, GSM1316325, GSM1316326, GSM1316332, GSM1316333</td>
</tr>
<tr class="even">
<td style="text-align: left;">RNA-seq</td>
<td style="text-align: left;">ERR266333, ERR266335, ERR266337, ERR266338, ERR266341, ERR266342, ERR266344, ERR266346, ERR266349, ERR266351</td>
</tr>
</tbody>
</table>
<h3 id="analysis-of-chip-seq-data-sets">Analysis of ChIP-seq Data Sets</h3>
<p>The raw reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in Bowtie version 1.0.0 <span class="citation" data-cites="RU8ikjzP">[<a href="#ref-RU8ikjzP">5</a>]</span>. The aligned reads are subsequently filtered for quality and uniquely mappable reads using Samtools version 0.1.19 <span class="citation" data-cites="IPc8cy7s">[<a href="#ref-IPc8cy7s">6</a>]</span> and Picard version 1.127 <span class="citation" data-cites="tfF98ztu">[<a href="#ref-tfF98ztu">7</a>]</span>. Library complexity is measured using BEDTools version 2.17.0 <span class="citation" data-cites="1HWiAHnIw">[<a href="#ref-1HWiAHnIw">8</a>]</span> and meet ENCODE data quality standards <span class="citation" data-cites="hvhWxir9">[<a href="#ref-hvhWxir9">9</a>]</span>. Relaxed peaks were called using MACS version 2.1.0 <span class="citation" data-cites="Bhecn2fS">[<a href="#ref-Bhecn2fS">10</a>]</span> with a p-value of <span class="math inline">\(1 \times 10^{-2}\)</span> for each replicate, pooled replicates’ reads and pseudoreplicates. Peak calls that are replicated from the pooled replicated that are either observed in both replicates, or in both pseudoreplicates are used for subsequent analysis.</p>
<h3 id="analysis-of-rna-seq-data-sets">Analysis of RNA-seq Data Sets</h3>
<p>The raw reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in STAR version 2.4.2a <span class="citation" data-cites="tTu8Ds9Z">[<a href="#ref-tTu8Ds9Z">11</a>]</span>. Quantification of genes against Gencode version 19) <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">12</a>]</span> annotations was done using default parameters in RSEM version 1.2.31 <span class="citation" data-cites="Dh2n1EV3">[<a href="#ref-Dh2n1EV3">13</a>]</span>.</p>
<h3 id="analysis-of-gro-seq-data">Analysis of GRO-seq Data</h3>
<p>The GRO-seq reads were trimmed to the first 36 bases, to trim adapter and low quality sequence, using default parameters of fastx_trimer in fastx-toolkit version 0.0.13.2 <span class="citation" data-cites="jwUQ55cX">[<a href="#ref-jwUQ55cX">14</a>]</span>. The trimmed reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in BWA version 0.7.12 <span class="citation" data-cites="roi5bwXL">[<a href="#ref-roi5bwXL">15</a>]</span>.</p>
<h3 id="kernel-density">Kernel Density</h3>
<p>Kernel density plot representations were used to express the univariate distribution of ChIP-seq reads under peaks, RNA-seq reads for protein-coding genes and GRO-seq reads for short paired and short unpaired eRNAs. The kernel density plots were calculated in Python (ver. 2.7.11) using the kdeplot function from <a href="http://seaborn.pydata.org/">seaborn</a> version 0.7.1 <span class="citation" data-cites="qxfTM222">[<a href="#ref-qxfTM222">16</a>]</span> with default parameters.</p>
<h3 id="defining-transcription-start-sites">Defining Transcription Start Sites</h3>
<p>We made distinct transcription start sites (TSS) for protein-coding genes from Gencode version 19 <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">12</a>]</span> annotations using MakeGencodeTSS <span class="citation" data-cites="tDy18Yol">[<a href="#ref-tDy18Yol">17</a>]</span>.</p>
<h3 id="enhancer-calling-by-gro-seq">Enhancer calling by GRO-seq</h3>
<h4 id="transcript-calling."><em>Transcript calling</em>.</h4>
<p>Transcript calling was performed using a two-state hidden Markov model using the groHMM data analysis package version 3.4 <span class="citation" data-cites="1BkIYUDLC 1HVy2rbI lPKPHCBB">[<a href="#ref-1BkIYUDLC">18</a>–<a href="#ref-lPKPHCBB">20</a>]</span> on each individual cell lines. The negative log transition probability of the switch between transcribed state to non-transcribed state and the variance in read counts in the non-transcribed state that are used to predict the transcription units for the cell lines are listed Table <a href="#tbl:groseq-tune">2</a>.</p>
<a name="tbl:groseq-tune"></a>
<table>
<caption>Table 2: <strong>groHMM tunning parameters.</strong> </caption>
<thead>
<tr class="header">
<th style="text-align: left;">Cell Line</th>
<th style="text-align: center;">-Log Transition Probability</th>
<th style="text-align: center;">Variance in read counts</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">hES</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">45</td>
</tr>
<tr class="even">
<td style="text-align: left;">DE</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">35</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GT</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">50</td>
</tr>
<tr class="even">
<td style="text-align: left;">FG</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">35</td>
</tr>
<tr class="odd">
<td style="text-align: left;">PE</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">35</td>
</tr>
</tbody>
</table>
<p>We then built a universe of transcripts by merging the groHMM-called transcripts from individual cell lines and stratifying the boundaries to remove overlaps/redundancies occurring from the union of all transcripts.</p>
<h4 id="calling-enhancer-transcripts."><em>Calling Enhancer Transcripts</em>.</h4>
<p>We filtered and collected a subset of short intergenic transcripts <span class="math inline">\(&lt;\)</span> 9 kb in length and <span class="math inline">\(&gt;\)</span> 3 kb away from known transcription start sites (TSSs) of protein-coding genes from Gencode version 19 annotations <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">12</a>]</span>, and H3K4me3 peaks. These were further classified into (1) short paired eRNAs and (2) short unpaired eRNAs as described previously <span class="citation" data-cites="1DE6cLccI">[<a href="#ref-1DE6cLccI">21</a>]</span>. For the short paired eRNAs, the sum of the GRO-seq RPKM values for both strands of DNA was used to call an enhancer transcript pair as expressed using a criterion of RPKM <span class="math inline">\(\geq\)</span> 0.5. For the short unpaired eRNAs, an RPKM cutoff of <span class="math inline">\(\geq\)</span> 1 was used to call an enhancer transcript as expressed. The universe of expressed eRNAs (short paired and short unpaired) was assembled using the cutoffs noted above for each cell line and was used for further analyses.</p>
<h4 id="motif-analyses."><em>Motif Analyses</em>.</h4>
<p>De novo motif analyses were performed on a 1 kb region (<span class="math inline">\(\pm\)</span> 500 bp) surrounding the peak summit or the transcription start site for short paired and short unpaired eRNAs, respectively, using the command-line version of MEME from MEME Suite version 4.11.1 <span class="citation" data-cites="dwj6qSn3">[<a href="#ref-dwj6qSn3">22</a>]</span>. The following parameters were used for motif prediction: (1) zero or one occurrence per sequence (-mod zoops); (2) number of motifs (-nmotifs 15); (3) minimum, maximum width of the motif (-minw 8, -maxw 15); and (4) search for motif in given strand and reverse complement strand (-revcomp). The predicted motifs from MEME were matched to known motifs in the JASPAR database (JASPAR_CORE_2016_vertebrates.meme) <span class="citation" data-cites="1DrZNoXOU">[<a href="#ref-1DrZNoXOU">23</a>]</span> using TOMTOM <span class="citation" data-cites="1C9AIanMe">[<a href="#ref-1C9AIanMe">24</a>]</span>.</p>
<h3 id="enhancer-calling-by-chip-seq">Enhancer calling by ChIP-seq</h3>
<h4 id="calling-active-enhancers."><em>Calling Active Enhancers</em>.</h4>
<p>We built a universe of peak calls by merging the peaks from individual cell lines for histone modifications (H3K4me1 and H3K27ac) and stratifying the boundaries to remove overlaps/redundancies occurring from the union of all peaks. Potential enhancers were defined as peaks that are <span class="math inline">\(&gt;\)</span> 3kb from known TSS, protein coding genes from Gencode version 19 annotations <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">12</a>]</span>, and H3K4me3 peaks. A RPKM cutoff of <span class="math inline">\(\geq\)</span> 1 of H3K4me1 and <span class="math inline">\(\geq\)</span> 1 H3K27ac in at least 1 cell line was used to call a peak as an active enhancer. The universe of active enhancers was assembled using the cutoffs noted above for each cell line and was used for further analyses.</p>
<h4 id="motif-analyses.-1"><em>Motif Analyses</em>.</h4>
<p>De novo motif analyses were performed on a 1 kb region (<span class="math inline">\(\pm\)</span> 500 bp) surrounding the peak summit for the top 10000 enhancers, using the command-line version of MEME-ChIP from MEME Suite version 4.11.1 <span class="citation" data-cites="dwj6qSn3 TfukIx2u">[<a href="#ref-dwj6qSn3">22</a>,<a href="#ref-TfukIx2u">25</a>]</span>. The following parameters were used for motif prediction: (1) zero or one occurrence per sequence (-mod zoops); (2) number of motifs (-nmotifs 15); (3) minimum, maximum width of the motif (-minw 8, -maxw 15). All other parameters were set at the default. The predicted motifs from MEME were matched to known motifs in the JASPAR database (JASPAR_CORE_2016_vertebrates.meme) <span class="citation" data-cites="1DrZNoXOU">[<a href="#ref-1DrZNoXOU">23</a>]</span> using TOMTOM <span class="citation" data-cites="1C9AIanMe">[<a href="#ref-1C9AIanMe">24</a>]</span>.</p>
<h3 id="generating-heatmaps-and-clusters">Generating Heatmaps and Clusters</h3>
<p>For each cell line, the functional scores were Z-score normalized. To identify cognate transcription factors by cell type, we performed hierarchical clustering by calculating the Euclidean distance using clustermap from <a href="http://seaborn.pydata.org/">seaborn</a> version 0.7.1 <span class="citation" data-cites="qxfTM222">[<a href="#ref-qxfTM222">16</a>]</span>. For visualization of the multidimensional TFSEE scores, we performed t-distributed stochastic neighbor embedding analysis (t-SNE) <span class="citation" data-cites="N7ngUHQX">[<a href="#ref-N7ngUHQX">26</a>]</span> using the TSNE function and labeled the clusters by calculating K-means clustering using the KMeans function with the expectation-maximization algorithm in <a href="http://scikit-learn.org/">scikit-learn</a> version 0.17.1 <span class="citation" data-cites="vHIKEjNs GGWEfSeb r8BJ7tJV">[<a href="#ref-vHIKEjNs">27</a>–<a href="#ref-r8BJ7tJV">29</a>]</span>.</p>
<h3 id="nearest-neighboring-gene-analyses-and-box-plots">Nearest Neighboring Gene Analyses and Box Plots</h3>
<p>The universe of expressed genes in each cell line was determined from the RNA-seq data using an FPKM cutoff <span class="math inline">\(&gt;\)</span> 0.4. The set of nearest neighboring expressed genes for each enhancer defined by an expressed eRNA or the enrichment of active histone marks was determined for each cell line. Box plot representations were used to express the levels of transcription or enrichment for each called enhancer and transcription of their nearest neighboring expressed genes. The read distribution (RPKM) for each enhancer or (FPKM) gene was calculated and plotted using the boxplot function from <a href="https://matplotlib.org/">matplotlib</a> version 2.0.2 <span class="citation" data-cites="1026Gxdsi oaruRtzO">[<a href="#ref-1026Gxdsi">30</a>,<a href="#ref-oaruRtzO">31</a>]</span>. Wilcoxon rank sum tests were performed to determine the statistical significance of all comparisons.</p>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-BTqz3DNL">
<p>1. <strong>Dynamic Chromatin Remodeling Mediated by Polycomb Proteins Orchestrates Pancreatic Differentiation of Human Embryonic Stem Cells</strong><br />
Ruiyu Xie, Logan J. Everett, Hee-Woong Lim, Nisha A. Patel, Jonathan Schug, Evert Kroon, Olivia G. Kelly, Allen Wang, Kevin A. D’Amour, Allan J. Robins, … Maike Sander<br />
<em>Cell Stem Cell</em> (2013-02) <a href="https://doi.org/10.1016/j.stem.2012.11.023" class="uri">https://doi.org/10.1016/j.stem.2012.11.023</a></p>
</div>
<div id="ref-18a6G1TE5">
<p>2. <strong>Epigenetic Priming of Enhancers Predicts Developmental Competence of hESC-Derived Endodermal Lineage Intermediates</strong><br />
Allen Wang, Feng Yue, Yan Li, Ruiyu Xie, Thomas Harper, Nisha A. Patel, Kayla Muth, Jeffrey Palmer, Yunjiang Qiu, Jinzhao Wang, … Maike Sander<br />
<em>Cell Stem Cell</em> (2015-04) <a href="https://doi.org/10.1016/j.stem.2015.02.013" class="uri">https://doi.org/10.1016/j.stem.2015.02.013</a></p>
</div>
<div id="ref-Bc3QGVe7">
<p>3. <strong>GEO</strong><br />
Gene Expression Omnibus<br />
<a href="https://www.ncbi.nlm.nih.gov/geo/" class="uri">https://www.ncbi.nlm.nih.gov/geo/</a></p>
</div>
<div id="ref-937jwJMM">
<p>4. <strong>ArrayExpress</strong><br />
ArrayExpress – functional genomics data<br />
<a href="http://www.ebi.ac.uk/arrayexpress/" class="uri">http://www.ebi.ac.uk/arrayexpress/</a></p>
</div>
<div id="ref-RU8ikjzP">
<p>5. <strong>Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</strong><br />
Ben Langmead, Cole Trapnell, Mihai Pop, Steven L Salzberg<br />
<em>Genome Biology</em> (2009) <a href="https://doi.org/10.1186/gb-2009-10-3-r25" class="uri">https://doi.org/10.1186/gb-2009-10-3-r25</a></p>
</div>
<div id="ref-IPc8cy7s">
<p>6. <strong>The Sequence Alignment/Map format and SAMtools</strong><br />
H. Li, B. Handsaker, A. Wysoker, T. Fennell, J. Ruan, N. Homer, G. Marth, G. Abecasis, R. Durbin, <br />
<em>Bioinformatics</em> (2009-06-08) <a href="https://doi.org/10.1093/bioinformatics/btp352" class="uri">https://doi.org/10.1093/bioinformatics/btp352</a></p>
</div>
<div id="ref-tfF98ztu">
<p>7. <strong>Picard</strong><br />
Broad Institute<br />
<em>GitHub</em> <a href="http://broadinstitute.github.io/picard/" class="uri">http://broadinstitute.github.io/picard/</a></p>
</div>
<div id="ref-1HWiAHnIw">
<p>8. <strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong><br />
Aaron R. Quinlan, Ira M. Hall<br />
<em>Bioinformatics</em> (2010-01-28) <a href="https://doi.org/10.1093/bioinformatics/btq033" class="uri">https://doi.org/10.1093/bioinformatics/btq033</a></p>
</div>
<div id="ref-hvhWxir9">
<p>9. <strong>ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia</strong><br />
S. G. Landt, G. K. Marinov, A. Kundaje, P. Kheradpour, F. Pauli, S. Batzoglou, B. E. Bernstein, P. Bickel, J. B. Brown, P. Cayting, … M. Snyder<br />
<em>Genome Research</em> (2012-09-01) <a href="https://doi.org/10.1101/gr.136184.111" class="uri">https://doi.org/10.1101/gr.136184.111</a></p>
</div>
<div id="ref-Bhecn2fS">
<p>10. <strong>Identifying ChIP-seq enrichment using MACS</strong><br />
Jianxing Feng, Tao Liu, Bo Qin, Yong Zhang, Xiaole Shirley Liu<br />
<em>Nature Protocols</em> (2012-08-30) <a href="https://doi.org/10.1038/nprot.2012.101" class="uri">https://doi.org/10.1038/nprot.2012.101</a></p>
</div>
<div id="ref-tTu8Ds9Z">
<p>11. <strong>STAR: ultrafast universal RNA-seq aligner</strong><br />
Alexander Dobin, Carrie A. Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, Thomas R. Gingeras<br />
<em>Bioinformatics</em> (2012-10-25) <a href="https://doi.org/10.1093/bioinformatics/bts635" class="uri">https://doi.org/10.1093/bioinformatics/bts635</a></p>
</div>
<div id="ref-15rkMH6SD">
<p>12. <strong>GENCODE: The reference human genome annotation for The ENCODE Project</strong><br />
J. Harrow, A. Frankish, J. M. Gonzalez, E. Tapanari, M. Diekhans, F. Kokocinski, B. L. Aken, D. Barrell, A. Zadissa, S. Searle, … T. J. Hubbard<br />
<em>Genome Research</em> (2012-09-01) <a href="https://doi.org/10.1101/gr.135350.111" class="uri">https://doi.org/10.1101/gr.135350.111</a></p>
</div>
<div id="ref-Dh2n1EV3">
<p>13. <strong>RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</strong><br />
Bo Li, Colin N Dewey<br />
<em>BMC Bioinformatics</em> (2011) <a href="https://doi.org/10.1186/1471-2105-12-323" class="uri">https://doi.org/10.1186/1471-2105-12-323</a></p>
</div>
<div id="ref-jwUQ55cX">
<p>14. <strong>FASTX-Toolkit</strong><br />
Hannon Lab<br />
<a href="http://hannonlab.cshl.edu/fastx_toolkit/" class="uri">http://hannonlab.cshl.edu/fastx_toolkit/</a></p>
</div>
<div id="ref-roi5bwXL">
<p>15. <strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong><br />
H. Li, R. Durbin<br />
<em>Bioinformatics</em> (2009-05-18) <a href="https://doi.org/10.1093/bioinformatics/btp324" class="uri">https://doi.org/10.1093/bioinformatics/btp324</a></p>
</div>
<div id="ref-qxfTM222">
<p>16. <strong>Seaborn: V0.7.1 (June 2016)</strong><br />
Michael Waskom, Olga Botvinnik, Drewokane, Paul Hobson, David, Yaroslav Halchenko, Saulius Lukauskas, John B. Cole, Jordi Warmenhoven, Julian De Ruiter, … Antony Lee<br />
<em>Zenodo</em> (2016-06-05) <a href="https://doi.org/10.5281/zenodo.54844" class="uri">https://doi.org/10.5281/zenodo.54844</a></p>
</div>
<div id="ref-tDy18Yol">
<p>17. <strong>MakeGenecodeTSS</strong><br />
Sarah Djebali<br />
<em>GitHub</em> <a href="https://github.com/sdjebali/MakeGencodeTSS" class="uri">https://github.com/sdjebali/MakeGencodeTSS</a></p>
</div>
<div id="ref-1BkIYUDLC">
<p>18. <strong>A Rapid, Extensive, and Transient Transcriptional Response to Estrogen Signaling in Breast Cancer Cells</strong><br />
Nasun Hah, Charles G. Danko, Leighton Core, Joshua J. Waterfall, Adam Siepel, John T. Lis, W. Lee Kraus<br />
<em>Cell</em> (2011-05) <a href="https://doi.org/10.1016/j.cell.2011.03.042" class="uri">https://doi.org/10.1016/j.cell.2011.03.042</a></p>
</div>
<div id="ref-1HVy2rbI">
<p>19. <strong>groHMM</strong><br />
Minho Chae Charles G. Danko<br />
<em>Bioconductor</em> (2017) <a href="https://doi.org/10.18129/b9.bioc.grohmm" class="uri">https://doi.org/10.18129/b9.bioc.grohmm</a></p>
</div>
<div id="ref-lPKPHCBB">
<p>20. <strong>groHMM: a computational tool for identifying unannotated and cell type-specific transcription units from global run-on sequencing data</strong><br />
Minho Chae, Charles G. Danko, W. Lee Kraus<br />
<em>BMC Bioinformatics</em> (2015-07-16) <a href="https://doi.org/10.1186/s12859-015-0656-3" class="uri">https://doi.org/10.1186/s12859-015-0656-3</a></p>
</div>
<div id="ref-1DE6cLccI">
<p>21. <strong>Enhancer transcripts mark active estrogen receptor binding sites</strong><br />
N. Hah, S. Murakami, A. Nagari, C. G. Danko, W. L. Kraus<br />
<em>Genome Research</em> (2013-05-01) <a href="https://doi.org/10.1101/gr.152306.112" class="uri">https://doi.org/10.1101/gr.152306.112</a></p>
</div>
<div id="ref-dwj6qSn3">
<p>22. <strong>MEME SUITE: tools for motif discovery and searching</strong><br />
T. L. Bailey, M. Boden, F. A. Buske, M. Frith, C. E. Grant, L. Clementi, J. Ren, W. W. Li, W. S. Noble<br />
<em>Nucleic Acids Research</em> (2009-05-20) <a href="https://doi.org/10.1093/nar/gkp335" class="uri">https://doi.org/10.1093/nar/gkp335</a></p>
</div>
<div id="ref-1DrZNoXOU">
<p>23. <strong>JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles</strong><br />
Anthony Mathelier, Oriol Fornes, David J. Arenillas, Chih-yu Chen, Grégoire Denay, Jessica Lee, Wenqiang Shi, Casper Shyr, Ge Tan, Rebecca Worsley-Hunt, … Wyeth W. Wasserman<br />
<em>Nucleic Acids Research</em> (2015-11-03) <a href="https://doi.org/10.1093/nar/gkv1176" class="uri">https://doi.org/10.1093/nar/gkv1176</a></p>
</div>
<div id="ref-1C9AIanMe">
<p>24. <strong>Quantifying similarity between motifs</strong><br />
Shobhit Gupta, John A Stamatoyannopoulos, Timothy L Bailey, William Noble<br />
<em>Genome Biology</em> (2007) <a href="https://doi.org/10.1186/gb-2007-8-2-r24" class="uri">https://doi.org/10.1186/gb-2007-8-2-r24</a></p>
</div>
<div id="ref-TfukIx2u">
<p>25. <strong>MEME-ChIP: motif analysis of large DNA datasets</strong><br />
Philip Machanick, Timothy L. Bailey<br />
<em>Bioinformatics</em> (2011-04-12) <a href="https://doi.org/10.1093/bioinformatics/btr189" class="uri">https://doi.org/10.1093/bioinformatics/btr189</a></p>
</div>
<div id="ref-N7ngUHQX">
<p>26. <strong>Visualizing data using t-SNE</strong><br />
Laurens van der Maaten, Geoffrey Hinton<br />
<em>Journal of Machine Learning Research 9</em> (2008-11)</p>
</div>
<div id="ref-vHIKEjNs">
<p>27. <strong>Scikit-learn: Machine Learning in Python</strong><br />
Fabian Pedregosa, Gaël Varoquaux, Alexandre Gramfort, Vincent Michel, Bertrand Thirion, Olivier Grisel, Mathieu Blondel, Gilles Louppe, Peter Prettenhofer, Ron Weiss, … Édouard Duchesnay<br />
<em>arXiv</em> (2012-01-02) <a href="https://arxiv.org/abs/1201.0490v3" class="uri">https://arxiv.org/abs/1201.0490v3</a></p>
</div>
<div id="ref-GGWEfSeb">
<p>28. <strong>Visualizing Large-scale and High-dimensional Data</strong><br />
Jian Tang, Jingzhou Liu, Ming Zhang, Qiaozhu Mei<br />
<em>Proceedings of the 25th International Conference on World Wide Web - WWW ’16</em> (2016) <a href="https://doi.org/10.1145/2872427.2883041" class="uri">https://doi.org/10.1145/2872427.2883041</a></p>
</div>
<div id="ref-r8BJ7tJV">
<p>29. <strong>Scikit-Learn: 0.17.1 Release Tag For Doi</strong><br />
Olivier Grisel, Andreas Mueller, Fabian Pedregosa, Lars, Alexandre Gramfort, Gilles Louppe, Peter Prettenhofer, Mathieu Blondel, Vlad Niculae, Arnaud Joly, … Maheshakya Wijewardena<br />
<em>Zenodo</em> (2016-04-17) <a href="https://doi.org/10.5281/zenodo.49911" class="uri">https://doi.org/10.5281/zenodo.49911</a></p>
</div>
<div id="ref-1026Gxdsi">
<p>30. <strong>Matplotlib: A 2D Graphics Environment</strong><br />
John D. Hunter<br />
<em>Computing in Science &amp; Engineering</em> (2007) <a href="https://doi.org/10.1109/mcse.2007.55" class="uri">https://doi.org/10.1109/mcse.2007.55</a></p>
</div>
<div id="ref-oaruRtzO">
<p>31. <strong>Matplotlib/Matplotlib V2.0.2</strong><br />
Michael Droettboom, Thomas A Caswell, John Hunter, Eric Firing, Jens Hedegaard Nielsen, Nelle Varoquaux, Benjamin Root, Phil Elson, Darren Dale, Jae-Joon Lee, … Nikita Kniazev<br />
<em>Zenodo</em> (2017-05-10) <a href="https://doi.org/10.5281/zenodo.573577" class="uri">https://doi.org/10.5281/zenodo.573577</a></p>
</div>
</div>
<script>
// AnchorJS minified version below.
// Source https://github.com/bryanbraun/anchorjs/blob/064abdd0987f305933ec4982af6d0c1cf2fd0814/anchor.js

/**
 * AnchorJS - v4.0.0 - 2017-06-02
 * https://github.com/bryanbraun/anchorjs
 * Copyright (c) 2017 Bryan Braun; Licensed MIT
 */
!function(A,e){"use strict";"function"==typeof define&&define.amd?define([],e):"object"==typeof module&&module.exports?module.exports=e():(A.AnchorJS=e(),A.anchors=new A.AnchorJS)}(this,function(){"use strict";function A(A){function e(A){A.icon=A.hasOwnProperty("icon")?A.icon:"",A.visible=A.hasOwnProperty("visible")?A.visible:"hover",A.placement=A.hasOwnProperty("placement")?A.placement:"right",A.class=A.hasOwnProperty("class")?A.class:"",A.truncate=A.hasOwnProperty("truncate")?Math.floor(A.truncate):64}function t(A){var e;if("string"==typeof A||A instanceof String)e=[].slice.call(document.querySelectorAll(A));else{if(!(Array.isArray(A)||A instanceof NodeList))throw new Error("The selector provided to AnchorJS was invalid.");e=[].slice.call(A)}return e}function n(){if(null===document.head.querySelector("style.anchorjs")){var A,e=document.createElement("style");e.className="anchorjs",e.appendChild(document.createTextNode("")),void 0===(A=document.head.querySelector('[rel="stylesheet"], style'))?document.head.appendChild(e):document.head.insertBefore(e,A),e.sheet.insertRule(" .anchorjs-link {   opacity: 0;   text-decoration: none;   -webkit-font-smoothing: antialiased;   -moz-osx-font-smoothing: grayscale; }",e.sheet.cssRules.length),e.sheet.insertRule(" *:hover > .anchorjs-link, .anchorjs-link:focus  {   opacity: 1; }",e.sheet.cssRules.length),e.sheet.insertRule(" [data-anchorjs-icon]::after {   content: attr(data-anchorjs-icon); }",e.sheet.cssRules.length),e.sheet.insertRule(' @font-face {   font-family: "anchorjs-icons";   src: url(data:n/a;base64,AAEAAAALAIAAAwAwT1MvMg8yG2cAAAE4AAAAYGNtYXDp3gC3AAABpAAAAExnYXNwAAAAEAAAA9wAAAAIZ2x5ZlQCcfwAAAH4AAABCGhlYWQHFvHyAAAAvAAAADZoaGVhBnACFwAAAPQAAAAkaG10eASAADEAAAGYAAAADGxvY2EACACEAAAB8AAAAAhtYXhwAAYAVwAAARgAAAAgbmFtZQGOH9cAAAMAAAAAunBvc3QAAwAAAAADvAAAACAAAQAAAAEAAHzE2p9fDzz1AAkEAAAAAADRecUWAAAAANQA6R8AAAAAAoACwAAAAAgAAgAAAAAAAAABAAADwP/AAAACgAAA/9MCrQABAAAAAAAAAAAAAAAAAAAAAwABAAAAAwBVAAIAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAMCQAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAg//0DwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAAIAAAACgAAxAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADAAAAAIAAgAAgAAACDpy//9//8AAAAg6cv//f///+EWNwADAAEAAAAAAAAAAAAAAAAACACEAAEAAAAAAAAAAAAAAAAxAAACAAQARAKAAsAAKwBUAAABIiYnJjQ3NzY2MzIWFxYUBwcGIicmNDc3NjQnJiYjIgYHBwYUFxYUBwYGIwciJicmNDc3NjIXFhQHBwYUFxYWMzI2Nzc2NCcmNDc2MhcWFAcHBgYjARQGDAUtLXoWOR8fORYtLTgKGwoKCjgaGg0gEhIgDXoaGgkJBQwHdR85Fi0tOAobCgoKOBoaDSASEiANehoaCQkKGwotLXoWOR8BMwUFLYEuehYXFxYugC44CQkKGwo4GkoaDQ0NDXoaShoKGwoFBe8XFi6ALjgJCQobCjgaShoNDQ0NehpKGgobCgoKLYEuehYXAAAADACWAAEAAAAAAAEACAAAAAEAAAAAAAIAAwAIAAEAAAAAAAMACAAAAAEAAAAAAAQACAAAAAEAAAAAAAUAAQALAAEAAAAAAAYACAAAAAMAAQQJAAEAEAAMAAMAAQQJAAIABgAcAAMAAQQJAAMAEAAMAAMAAQQJAAQAEAAMAAMAAQQJAAUAAgAiAAMAAQQJAAYAEAAMYW5jaG9yanM0MDBAAGEAbgBjAGgAbwByAGoAcwA0ADAAMABAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAP) format("truetype"); }',e.sheet.cssRules.length)}}this.options=A||{},this.elements=[],e(this.options),this.isTouchDevice=function(){return!!("ontouchstart"in window||window.DocumentTouch&&document instanceof DocumentTouch)},this.add=function(A){var i,o,s,c,r,a,h,l,u,d,f,g,p=[];if(e(this.options),"touch"===(g=this.options.visible)&&(g=this.isTouchDevice()?"always":"hover"),A||(A="h2, h3, h4, h5, h6"),0===(i=t(A)).length)return this;for(n(),o=document.querySelectorAll("[id]"),s=[].map.call(o,function(A){return A.id}),r=0;r<i.length;r++)if(this.hasAnchorJSLink(i[r]))p.push(r);else{if(i[r].hasAttribute("id"))c=i[r].getAttribute("id");else if(i[r].hasAttribute("data-anchor-id"))c=i[r].getAttribute("data-anchor-id");else{u=l=this.urlify(i[r].textContent),h=0;do{void 0!==a&&(u=l+"-"+h),a=s.indexOf(u),h+=1}while(-1!==a);a=void 0,s.push(u),i[r].setAttribute("id",u),c=u}d=c.replace(/-/g," "),(f=document.createElement("a")).className="anchorjs-link "+this.options.class,f.href="#"+c,f.setAttribute("aria-label","Anchor link for: "+d),f.setAttribute("data-anchorjs-icon",this.options.icon),"always"===g&&(f.style.opacity="1"),""===this.options.icon&&(f.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(f.style.lineHeight="inherit")),"left"===this.options.placement?(f.style.position="absolute",f.style.marginLeft="-1em",f.style.paddingRight="0.5em",i[r].insertBefore(f,i[r].firstChild)):(f.style.paddingLeft="0.375em",i[r].appendChild(f))}for(r=0;r<p.length;r++)i.splice(p[r]-r,1);return this.elements=this.elements.concat(i),this},this.remove=function(A){for(var e,n,i=t(A),o=0;o<i.length;o++)(n=i[o].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[o]))&&this.elements.splice(e,1),i[o].removeChild(n));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var t=/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\]/g;return this.options.truncate||e(this.options),A.trim().replace(/\'/gi,"").replace(t,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&(" "+A.firstChild.className+" ").indexOf(" anchorjs-link ")>-1,t=A.lastChild&&(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ")>-1;return e||t||!1}}return A});

// Enable links for selected headers
var anchors = new AnchorJS();
anchors.add("h2, h3, h4")
</script>
</body>
</html>
