## Background {.page_break_before}

Enhancers and the transcription factors (TFs) regulating their formation have been shown to play an important role in cell type-specific activation of gene expression [@doi:10.1038/nrg3682; @doi:10.1038/nrm3949].
Although thousands of potential enhancers have been identified in cell lines and tissues, identification of TFs driving active enhancer formation in each cell type remains a major challenge [@doi:10.1038/nature11247; @doi:10.1038/nature14248].

Active enhancers have been shown to share several common features; such as open and accessible regions of chromatin (as measured by DNase-seq or ATAC-seq) [@doi:10.1101/gr.4074106; @doi:10.1101/gr.152140.112; @doi:10.1371/journal.pgen.1004994] and post-translational modification of histone tails (as assessed by ChIP-seq), including H3K4me1 and H3K27ac [@doi:10.1038/ng1966; @doi:10.1038/nature07829; @doi:10.1534/g3.114.013565]. While these features successfully define many enhancers, recent genomic assays have shown enhancers tend to be bound by RNA polymerase II (Pol II) and are actively transcribed, producing enhancer RNAs (‘eRNAs’) [@doi:10.1371/journal.pbio.1000384; @doi:10.1016/j.cell.2015.08.008; @doi:10.1016/j.cell.2011.03.042]. Enhancer transcription (as measured by GRO-seq or PRO-seq) has
been shown to be used for enhancer prediction and track enhancer activity [@doi:10.1016/j.cell.2011.03.042; @doi:10.1038/nature10006; @doi:10.1101/gr.152306.112; @doi:10.1038/ng.3142; @doi:10.1186/s12859-015-0656-3; @doi:10.1038/nrm3949; @doi:10.1038/nature12210; @doi:10.1016/j.molcel.2015.02.001; @doi:10.1016/j.cell.2015.08.008].

In recent years, advances in technology have facilitated the large scale functional characterization of enhancer activity [@doi:10.1093/nar/gkl822; @doi:10.1101/gr.144899.112; @doi:10.1126/science.1232542; @doi:10.1101/gr.173518.114] and annotation of genome-wide binding sites of TFs in various cell types and tissues [@doi:10.1038/nature11247; @doi:10.1038/nature11245]. However, due to countless cell types, experimental conditions and the large number of TFs [@doi:10.1038/nrg2538], an integration of these independent methods
to study gene expression may not be achievable. Furthermore, analyses predicting TF binding sites (TFBSs), 4-12 nucleotides [@doi:10.1038/nature15518], utilizing databases of binding motifs [@doi:10.1016/j.cell.2012.12.009; @doi:10.1093/nar/gkv1176; @doi:10.1186/gb-2007-8-2-r24] to predict the most likely bound TFs fail to consider that such sequences frequently occur by chance in the genome and that TF occupation is cell type specific [@doi:10.1186/gb-2012-13-9-r50]. To overcome these limitations, we have in this work established a novel method, Total Functional Score of Enhancer Elements (TFSEE), which can identify cell type-specific enhancers and their cognate TFs.

In TFSEE, we integrate enhancer location and activity, TF motif prediction for each enhancer and the level of TF expression (Figure @fig:overview_tfsee). We have previously demonstrated TFSEE in the identification of
key breast cancer subtype-specific transcription factors determining growth outcomes [TODO:LONESTAR Reference]. In the studies presented herein, we demonstrate the broader use of TFSEE to identify transcription factors during the differentiation of embryonic stem cells into pancreatic cells.


Using TFSEE, we have identified key breast cancer subtype-specific transcription factors that are bound at active enhancers and dictate gene expression patterns determining growth outcomes. To demonstrate the broader utility of our approach, we have used this algorithm to identify transcription factors during the differentiation of embryonic stem cells into pancreatic cells.
Taken together our results show that TFSEE can be used to perform multilayer genomic data integration to uncover novel cell type-specific transcription factors that control lineage-specific enhancers (Figure {@fig:enhancer_predictions}A).
