## Background {.page_break_before}

Enhancers and the transcription factors (TFs) regulating their formation have been shown to play an important role in cell type-specific activation of gene expression [@doi:10.1038/nrg3682; @doi:10.1038/nrm3949].
Although thousands of potential enhancers have been identified in cell lines and tissues, identification of the enhancers that are active versus not active or poised remains remains a major challenge [@doi:10.1093/bib/bbv101].
In addition, the ability to identify the TFs acting at the numerous enhancers in each cell type is technically challenging [@doi:10.1038/nature11247; @doi:10.1038/nature14248].

Active enhancers have been shown to share several common features; such as increased chromatin accessibility (as measured by DNase-seq or ATAC-seq) [@doi:10.1101/gr.4074106; @doi:10.1101/gr.152140.112; @doi:10.1371/journal.pgen.1004994] and enrichment of post-translational modification of histone tails (as assessed by ChIP-seq), including H3K4me1 and H3K27ac [@doi:10.1038/ng1966; @doi:10.1038/nature07829; @doi:10.1534/g3.114.013565]. While these epigenetic features can successfully identify the location of many enhancers across the genome, they cannot readily differentiate between active and non-active enhancers [@doi:10.1093/nar/gkv1530; @doi:10.1186/s13072-015-0009-5]. However, recent genomic assays have shown that enhancers tend to be bound by RNA polymerase II (Pol II) and transcribed, producing non-coding RNAs known as enhancer RNAs (‘eRNAs’) [@doi:10.1371/journal.pbio.1000384; @doi:10.1016/j.cell.2015.08.008; @doi:10.1016/j.cell.2011.03.042].
While the functions of the enhancer RNA transcripts are unknown, we and others have shown that enhancer transcription (as measured by total RNA-seq, GRO-seq or PRO-seq) can be used in the absence of any other genomic information to as a predictor of enhancer activity [@doi:10.1016/j.cell.2011.03.042; @doi:10.1371/journal.pgen.1004610; @doi:10.1038/nature10006; @doi:10.1101/gr.152306.112; @doi:10.1038/ng.3142; @doi:10.1186/s12859-015-0656-3; @doi:10.1038/nrm3949; @doi:10.1038/nature12210; @doi:10.1016/j.molcel.2015.02.001; @doi:10.1016/j.cell.2015.08.008].

In recent years, advances in technology have facilitated the large scale functional characterization of enhancer activity [@doi:10.1093/nar/gkl822; @doi:10.1101/gr.144899.112; @doi:10.1126/science.1232542; @doi:10.1101/gr.173518.114] and annotation of genome-wide binding sites of TFs in various cell types and tissues [@doi:10.1038/nature11247; @doi:10.1038/nature11245]. However, due to countless cell types, experimental conditions and the large number of TFs [@doi:10.1038/nrg2538], integration of these independent data sets to achieve a comprehensive analysis of gene expression and actionable predictions of TFs driving cell type-specific gene expression can be very challenging. Furthermore, analyses that predict TF binding sites (TFBSs), which are usually 4-12 nucleotides in length [@doi:10.1038/nature15518], using TF binding profile databases [@doi:10.1016/j.cell.2012.12.009; @doi:10.1093/nar/gkv1176; @doi:10.1186/gb-2007-8-2-r24], fail to consider that such sequences occur frequently by chance throughout the genome and that TF binding is cell type specific [@doi:10.1186/gb-2012-13-9-r50]. To overcome these limitations, we established a novel method called Total Functional Score of Enhancer Elements (TFSEE), which can be used to identify location and activity of enhancers in any cell or tissue type together with their cognate TFs.

In TFSEE, we integrate enhancer location and activity, TF motif prediction for each enhancer and the level of TF expression (Figure @fig:overview_tfsee). We have previously demonstrated TFSEE in the identification of
key breast cancer subtype-specific transcription factors determining growth outcomes [@doi:10.1101/gr.226019.117]. In the studies presented herein, we demonstrate the broader use of TFSEE to identify enhancers and TFs during the differentiation of embryonic stem cells into pancreatic progenitor cells.
Taken together our results show that TFSEE can be used to perform multilayer genomic data integration to uncover novel cell type-specific transcription factors that control lineage-specific enhancers (Figure {@fig:enhancer_predictions}A).
