## Background

Enhancers and the transcription factors (TFs) regulating their formation have been shown to play an important role in cell type-specific activation of gene expression [@doi:10.1038/nrg3682; @doi:10.1038/nrm3949].
Although thousands of potential enhancers have been identified in cell lines and tissues, identification of TFs driving active enhancer formation in each cell type remains a major challenge [@doi:10.1038/nature11247; @doi:10.1038/nature14248].

Active enhancers have been shown to share several common features; such as open and accessible regions of chromatin (as measured by DNase-seq or ATAC-seq) [@doi:10.1101/gr.4074106; @doi:10.1101/gr.152140.112; @doi:10.1371/journal.pgen.1004994] and post-translational modification of histone tails (as assessed by ChIP-seq), including H3K4me1 and H3K27ac [@doi:10.1038/ng1966; @doi:10.1038/nature07829; @doi:10.1534/g3.114.013565]. While these features successfully define many enhancers, recent genomic assays have shown enhancers tend to be bound by RNA polymerase II (Pol II) and are actively transcribed, producing enhancer RNAs (‘eRNAs’) [@doi:10.1371/journal.pbio.1000384; @doi:10.1016/j.cell.2015.08.008; @doi:10.1016/j.cell.2011.03.042]. Enhancer transcription (as measured by GRO-seq or PRO-seq) has
been shown to be used for enhancer prediction and track enhancer activity [@doi:10.1016/j.cell.2011.03.042; @doi:10.1038/nature10006; @doi:10.1101/gr.152306.112; @doi:10.1038/ng.3142; @doi:10.1186/s12859-015-0656-3; @doi:10.1038/nrm3949; @doi:10.1038/nature12210; @doi:10.1016/j.molcel.2015.02.001; @doi:10.1016/j.cell.2015.08.008].

In recent years, advances in technology have facilitated the large scale functional characterization of enhancer activity [@doi:10.1101/gr.144899.112; @doi:10.1126/science.1232542; @doi:10.1101/gr.173518.114], annotation of genome-wide binding sites of TFs [@doi:10.1038/nature11247; @doi:10.1038/nature11245] and identification of binding motifs for hundreds of TFs [@doi:10.1016/j.cell.2012.12.009]. However,


Here, we propose TFSEE to identify cell type-specific TFs the promote lineage-specific enhancers. TFSEE integrates the magnitude of enhancer transcription (GRO-seq), enrichment of marks associated with enhancers (H3K4me1 and H3K27ac ChIP-seq), TF mRNA expression levels (RNA-seq), and TF motif p-values (MEME).
