## Results

### The TFSEE model

The TFSEE model integrates multiple genomics assays, GRO-seq, RNA-seq, and ChIP-seq, data with TF motif information to predict TFs driving the formation of active enhancers and the locations of their cognate enhancers.
The TFSEE model consists of five key data processing steps (Figure {@fig:tfsee_processing}) followed by a data integration stage (Figure {@fig:overview_tfsee}).
In step 1 of TFSEE, a universe of active enhancers across the different constituent cell types are identified. The enhancers can be identified either by enhancer transcription (GRO-seq or total RNA-seq) (Figure {@fig:enhancer_pipeline}A) or enrichment of epigenomic marks (H3K4me1 and H3K27ac) (Figure {@fig:enhancer_pipeline}B).
In step 2 of TFSEE, genome-wide enhancer activity levels are assessed by calculating the enrichment (H3K4me1 and H3K27ac) and eRNA transcription (GRO-seq or total RNA-seq) profiles under the universe of enhancers per cell type.
TFSEE was designed to detect enhancer activity changes and TF:enhancer links for each cell type.
All TF to enhancer links are determined by a de novo motif search and summarizing the probability of that TF using the tools in steps 3-4 of TFSEE, which creates a table annotating enhancer to TF for each cell type.
For all TFs identified TFSEE calculates the expression profile using (GRO-seq or RNA-seq) across every cell type in step 5.

The final stage integrates all the data in steps 1-5 (Figure {@fig:overview_tfsee}) to determine TFSEE score matrix and heatmap.
First, we generate an enhancer activity matrix A<sub>CxE</sub> for all cell types C for the universe active of E enhancers, as determined from step 2.
We assume that the enhancer activity of each cell type is linearly correlated to the amount enhancer transcription (GRO-seq or total RNA-seq, G), and to the epigenomic marks (H3K4me1, M and H3K27ac, H).
To reduce bias each individual enhancer enrichment is scaled between 0 and 1.
Enhancer activity can be expressed as the following formula:

<center>$A = G + M + H$</center>

Next, the enhancer activity matrix A<sub>CxE</sub>, is combined with motif prediction matrix T<sub>ExF</sub>, represent scaled motif prediction p-values, T, for each enhancer E, to form an intermediate matrix product. This matrix product is entrywise combined with TF expression matrix R, from step 5, the expression of each TF F for each cell type C, into a resulting matrix Z composed of C cell types and F TFs.
TFSEE can be expressed as the following formula:

<center>$Z =( A \times T) \circ  R$</center>

### Choice of biological model system and data

To better understand the TF-driven transcriptional programs using TFSEE, we used previously published transcriptional and epigenomic data from time course differentiation of human embryonic stem cells (hESC) towards pancreatic cell type [@doi:10.1016/j.stem.2012.11.023; @doi:10.1016/j.stem.2015.02.013] (Figure {@fig:enhancer_predictions}A).
For these analyses, we used GRO-seq and RNA-seq, as well as ChIP-seq for 3 different histone modifications at five defined stages of differentiation: hESCs, definitive endoderm (DE), primitive gut tube (GT), posterior foregut (FG), and pancreatic endoderm (PE) (Figure {@fig:enhancer_predictions}A, Table {@tbl:data-sets}).
This embryonic development model allows us to explore the spatiotemporal gene regulation during development, by enhancers and TFs.

### Unbiased Identification of Enhancers during Pancreatic Differentiation

We and other have shown that enhancers can be identified using enrichment of histone modifications (e.g. H3K4me1 and H3K27ac) [@doi:10.1038/ng1966; @doi:10.1038/nature07829; @doi:10.1534/g3.114.013565] or by enhancer transcription [@doi:10.1016/j.cell.2011.03.042].
We used a computational pipeline to identify a universe of eRNA transcripts from GRO-seq (Figure {@fig:enhancer_pipeline}A) or enrichment of epigenomic marks (H3K4me1 or H3K27ac) (Figure {@fig:enhancer_pipeline}B) for the cell lines
in the pancreatic differentiation time course model.
All potential enhancers were filtered to be $>$ 3 kb away from known transcription start sites (TSSs) of protein-coding genes from Gencode version 19 annotations [@doi:10.1101/gr.135350.111], and active promoters, as identified by H3K4me3 [@doi:10.1016/j.cell.2007.05.042] (Figure {@fig:density_plot}A) to avoid complications in the analysis associated with overlapping promoter transcription.

Using enrichment of H3K4me1 and H3K27ac, RPKM cutoff of $\geq$ 1 (Figure {@fig:density_plot}B and C) in at least one cell line, we determined there to be set of 182,335 candidate enhancers across all stages of pancreatic differentiation (Figure {@fig:enhancer_predictions}B).
We categorized these candidate enhancers for each cell line and found that $\le$ 20% of the enhancers, as determined by presences of H3K4me1 and H3K27ac, are active in each cell line and the majority are marked by only H3K4me1 (Figure {@fig:enhancer_predictions}B).
These results confirm the enhancer landscape across pancreatic differentiation reported by Wang *et al*.
We then identified a set of 4,974 candidate enhancers (Figure {@fig:enhancer_predictions}B) by GRO-seq as described previously [@doi:10.1101/gr.152306.112], using RPKM $\geq$ 0.5 or $\geq$ 1 (Figure {@fig:density_plot}D and E) in at least one cell line.
Compared to active enhancer by histone modifications, we found that the number of active enhancers in each cell line ranged from 77-25% of all candidate enhancers.

We compared the overlap from histone enhancer prediction methods (H3K4me1 or H3K27ac) to output from an enhancer transcription based approach (GRO-seq).
We found that 12% of enhancers called based on enhancer transcription using GRO-seq data are identified by all the of the other methods (enrichment of H3K4me1 and H3K27ac) (Figure {@fig:enhancer_predictions}C, {@fig:genome_browser}A).
In contrast, greater than 75% of the enhancers were solely identified by enhancer transcription (Figure {@fig:enhancer_predictions}C and D, {@fig:genome_browser}B).
Although H3K27ac and H3K4me1 might be two histone modifications commonly associated with enhancers, these are not the only chromatin mark involved and other modifications may be present that were not assayed for [@doi:10.1093/nar/gkt826].
Additionally, less than 1% of enhancers called based on enrichment of H3K4me1 or H3K27ac are identified by the other methods (Figure {@fig:enhancer_predictions}C).
This may be due, in part, to the fact that enhancer calling based on H3K4me1 or H3K27ac enrichment, yields much larger numbers of putative enhancers (Figure {@fig:enhancer_predictions}D), many of which may be false positives or inactive as the true regulatory elements (Figure {@fig:genome_browser}C and D). Nonetheless, as we show below, incorporating enhancer transcription into an TFSEE pipeline that includes information about H3K4me1 and H3K27ac enrichment, improves the identification cell type-specific enhancers.

### TFSEE identifies cell type-specific enhancers and their cognate TFs

We used the enhancer calls by Figure {@fig:enhancer_predictions}B, to identify cell type-specific enhancers and their cognate TFs, using TFSEE, either by enhancer transcription or enrichment of epigenomic marks.
We visualized the results from TFSEE using unsupervised hierarchical clustering, which grouped the cell types into two major clades: (1) FG, and PE (2) hESC, DE, and GT (Figure {@fig:tfsee_groseq}A, {@fig:tfsee_histone}A).
To better understand the TF:enhancer dynamics across all differentiation stages we clustered the TSEE score across all differentiation stages, revealing four major categories using enhancer transcription (Figure {@fig:tfsee_groseq}B).
We examined the enrichment of putative enhancers and their associated TFs across stages by quantifying their normalized TFSEE score. This analysis revealed four major clusters: 1. driving early (hESC, DE) and late pancreatic differentiation (FG and PE), 2. enriched in GT, 3. driving pre-pancreatic linage (hESC, DE and GT), and 4. driving late-pancreatic differentiation (FG and PE) (Figure {@fig:tfsee_groseq}D). In contrast, using only histone enrichment to identify enhancers, we retrieve only three clusters (Figure {@fig:tfsee_histone}B). These results highlight TF:enhancers driving pre-pancreatic lineage (hESC, DE and GT), and late-pancreatic differentiation (FG and PE), but fails to highlight any other stage specific drivers (Figure {@fig:tfsee_histone}C). We were particularly interested in TFs and enhancers that provided a clear demarcation of enrichment between pre- and late- pancreatic differentiation.

To investigate the the distinct roles of lineage specific enhancers and their cognate TFs, we first examined the mRNA levels of the corresponding predicted TFs of each cluster in each of the stages.
Our analysis revealed that TFs identified in pre-pancreatic lineage show equal expression across stages, while late-pancreatic TFs are highly expressed in FG and PE (Figure {@fig:late_pre_diff}A, {@fig:late_pre_histone}A) coinciding with pancreatic induction at the FG stage (Figure {@fig:enhancer_predictions}A).
Conversely, we didn't see an enrichment of TFs in a stage specific manner for either TFs enriched early (hESC, DE) and late pancreatic differentiation (FG and PE) or those maintaining GT pluripotency (Figure {@fig:primitive_gut}A).

Next, we determined if enhancer transcription corresponding to the enriched TFs, using TFSEE score, and the regulation of their nearby genes might regulate differentiation biology.
To do so, we identified the enhancers corresponding to the predicted TFs using enriched binding motif prediction, and then determined the level of transcription for each enhancer, using GRO-seq or H3K27ac ChIP-seq, (Figure {@fig:late_pre_diff}B, {@fig:late_pre_histone}B, {@fig:primitive_gut}B) and the nearest neighboring gene (upstream or downstream), using RNA-seq (Figure {@fig:late_pre_diff}C, {@fig:late_pre_histone}C, {@fig:primitive_gut}C).
Interestingly, transcribed enhancers exhibited stage specific enrichment, which doesn't correspond to the patterns found from TFSEE enrichment (Figure {@fig:late_pre_diff}B, {@fig:late_pre_histone}B, {@fig:primitive_gut}B).
This result reflects that 48% - 99% of the enhancers are shared between clusters and the variation between
clusters is due to differences in TF expression and affinity to motifs.
Likewise, the nearest neighboring gene for each transcribed enhancer doesn't exhibit stage specific enrichment (Figure {@fig:late_pre_diff}C, {@fig:late_pre_histone}C, {@fig:primitive_gut}C) due to the vast abundance of enhancers and thus
neighboring genes shared between the clusters.
However, without further high-throughput data to study promoter-enhancer linking (as measured by 4C, ChIA-PET, or Hi-C) [@doi:10.1038/ng1896, @doi:10.1016/j.ymeth.2012.08.009, @doi:10.1038/ng.3286]
it is difficult to understand the stage specific regulatory network.

To further understand the potential regulators of each cluster we determined a rank order frequency distribution for all TFs within each cluster (Figure {@fig:late_pre_diff}D and E).
This analysis revealed  enrichment of HINFP, RARG, ZIC3, and SP1-like family TFs (SP1 and SP8) which are important regulators of embryonic development [@doi:10.1073/pnas.0905651106, @doi:10.3858/emm.2009.41.8.091, @doi:10.1091/mbc.E06-07-0624, @doi:10.1111/j.1440-169X.2005.00797.x] (Figure {@fig:late_pre_diff}D).
Additionally, the Onecut family (ONECUT2 and ONECUT3), EGR1, MITF and FOXP1 TFs are enriched in cluster 4 and have been shown to function in pancreatic and islet cell development [@doi:10.1016/j.ydbio.2007.02.027, @doi:10.1210/en.2012-1064, @doi:10.2337/db12-1464, @doi:10.1007/s00125-015-3635-3] (Figure {@fig:late_pre_diff}E).
Similar results were obtained for comparison of TFSEE scores from different clusters using GRO-seq and Histone called enhancers (Figure {@fig:primitive_gut}D and E, {@fig:late_pre_histone}D and E).

To compare TFs enriched between GRO-seq and Histone driven TFSEE scores we compared the TFs enriched in pre- and late-pancreatic differentiation.
We found 9 and 12 TFs enriched in common for pre- and late-pancreatic respectively.
The differences in TFs that are enriched may be due, in part, to enhancer calling using H3K4me1 and H3K27ac enrichment, yields much larger numbers of putative enhancers, many of which may be false positives or inactive as true regulatory elements
producing more varied enriched TF motifs.
Altogether, our results show that TFSEE can be used to identify cell type specific TFs that control the biology of those cell types.
