## Material and Methods

### Genomic Data Curation

We used previously published GRO-seq, ChIP-seq and RNA-seq data from [@doi:10.1016/j.stem.2012.11.023; @doi:10.1016/j.stem.2015.02.013] of time course differentiation of human embryonic stem cells (hESC) to pancreatic endoderm (PE). All data sets are available from NCBI’s Gene Expression Omnibus [@url:https://www.ncbi.nlm.nih.gov/geo/] or EMBL-EBI’s ArrayExpress [@url:http://www.ebi.ac.uk/arrayexpress/] repositories using the accession numbers listed in Table @tbl:data-sets.

| Assay | Accessions |
| :--- | :-------- |
| GRO-seq | GSM1316306, GSM1316313, GSM1316320, GSM1316327, GSM1316334 |
| H3K4me3 ChIP-seq | ERR208008, ERR208014, ERR207998, ERR20798, ERR207999 |
| H3K4me1 ChIP-seq | GSM1316302, GSM1316303, GSM1316309, GSM1316316, GSM1316317, GSM1316310, GSM1316323, GSM1316324, GSM1316330, GSM1316331 |
| H3K27ac ChIP-seq | GSM1316300, GSM1316301, GSM1316307, GSM1316308, GSM1316314, GSM1316315, GSM1316321, GSM1316322, GSM1316328, GSM1316329 |
| Input ChIP-seq | ERR208001, ERR208012, ERR207984, ERR208011, ERR207986, GSM1316304, GSM1316305, GSM1316311, GSM1316312, GSM1316318, GSM1316319, GSM1316325, GSM1316326, GSM1316332, GSM1316333 |
| RNA-seq | ERR266333, ERR266335, ERR266337, ERR266338, ERR266341, ERR266342, ERR266344, ERR266346, ERR266349, ERR266351 |

Table: **Description and accession numbers of GRO-seq, ChIP-seq and RNA-seq datasets.**
{#tbl:data-sets}

### Analysis of ChIP-seq Data Sets

The raw reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in Bowtie (ver. 1.0.0) [@doi:10.1186/gb-2009-10-3-r25]. The aligned reads are subsequently filtered for quality and uniquely mappable reads using Samtools (ver. 0.1.19) [@doi:10.1093/bioinformatics/btp352] and Picard (ver. 1.127) [@url:http://broadinstitute.github.io/picard/]. Library complexity is measured using BEDTools (v 2.17.0) [@doi:10.1093/bioinformatics/btq033] and meet ENCODE data quality standards [@doi:10.1101/gr.136184.111]. Relaxed peaks were called using MACS (v2.1.0) [@doi:10.1038/nprot.2012.101] with a p-value of $1 \times 10^{-2}$ for each replicate, pooled replicates’ reads and pseudoreplicates. Peak calls that are replicated from the pooled replicated that are either observed in both replicates, or in both pseudoreplicates are used for subsequent analysis.

### Analysis of RNA-seq Data Sets

The raw reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in STAR (v2.4.2a) [@doi:10.1093/bioinformatics/bts635]. Quantification of genes against Gencode (v.19) [@doi:10.1101/gr.135350.111] annotations was done using default parameters in RSEM (v 1.2.31) [@doi:10.1186/1471-2105-12-323].

### Analysis of GRO-seq Data

The GRO-seq reads were trimmed to the first 36 bases, to trim adapter and low quality sequence, using default parameters of fastx_trimer in fastx-toolkit (v.0.0.13.2) [@url:http://hannonlab.cshl.edu/fastx_toolkit/].  The trimmed reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in BWA (v0.7.12) [@doi:10.1093/bioinformatics/btp324].

### Kernel Density

Kernel density plot representations were used to express the univariate distribution of ChIP-seq reads under peaks, RNA-seq reads for protein-coding genes and GRO-seq reads for short paired and short unpaired eRNAs.  The kernel density plots were calculated in Python (ver. 2.7.11) using the kdeplot function from seaborn libary [@url:http://seaborn.pydata.org/] with default parameters.

### Defining Transcription Start Sites

We made distinct transcription start sites (TSS) for protein-coding genes from Gencode (v.19) [@doi:10.1101/gr.135350.111] annotations using MakeGencodeTSS [@url:https://github.com/sdjebali/MakeGencodeTSS].  

### Enhancer calling by GRO-seq

#### *Transcript calling*.  

Transcript calling was performed using a two-state hidden Markov model using the groHMM data analysis package [@doi:10.1016/j.cell.2011.03.042;@doi:10.18129/B9.bioc.groHMM;@doi:10.1186/s12859-015-0656-3] on each individual cell lines. The negative log transition probability of the switch between transcribed state to non-transcribed state and the variance in read counts in the non-transcribed state that are used to predict the transcription units for the cell lines are listed Table @tbl:groseq-tune.

| Cell Line | -Log Transition Probability | Variance in read counts
| :--- | :--------: | :--------: |
| hES | 50 | 45 |
| DE | 50 | 35 |
| GT | 50 | 50 |
| FG | 50 | 35 |
| PE | 50 | 35 |

Table: **groHMM tunning parameters.**
{#tbl:groseq-tune}

We then built a universe of transcripts by merging the groHMM-called transcripts from individual cell lines and stratifying the boundaries to remove overlaps/redundancies occurring from the union of all transcripts.

#### *Calling Enhancer Transcripts*.  

We filtered and collected a subset of short intergenic transcripts $<$ 9 kb in length and $>$ 3 kb away from known transcription start sites (TSSs) of protein-coding genes from Gencode (v.19) annotations [@doi:10.1101/gr.135350.111], and H3K4me3 peaks.  These were further classified into (1) short paired eRNAs and (2) short unpaired eRNAs as described previously [@doi:10.1101/gr.152306.112].  For the short paired eRNAs, the sum of the GRO-seq RPKM values for both strands of DNA was used to call an enhancer transcript pair as expressed using a criterion of RPKM $\geq$ 0.5.  For the short unpaired eRNAs, an RPKM cutoff of $\geq$ 1 was used to call an enhancer transcript as expressed.  The universe of expressed eRNAs (short paired and short unpaired) was assembled using the cutoffs noted above for each cell line and was used for further analyses.

#### *Motif Analyses*.

De novo motif analyses were performed on a 1 kb region ($\pm$ 500 bp) surrounding the peak summit or the transcription start site for short paired and short unpaired eRNAs, respectively, using the command-line version of MEME (ver 4.11.1) [@doi:10.1093/nar/gkp335].  The following parameters were used for motif prediction: (1) zero or one occurrence per sequence (-mod zoops); (2) number of motifs (-nmotifs 15); (3) minimum, maximum width of the motif (-minw 8, -maxw 15); and (4) search for motif in given strand and reverse complement strand (-revcomp).  The predicted motifs from MEME were matched to known motifs in the JASPAR database (JASPAR_CORE_2016_vertebrates.meme) [@doi:10.1093/nar/gkv1176] using TOMTOM [@doi:10.1186/gb-2007-8-2-r24].
