<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Venkat S. Malladi" />
  <meta name="author" content="Anusha Nagari" />
  <meta name="author" content="Hector L. Franco" />
  <meta name="author" content="W. Lee Kraus" />
  <meta name="dcterms.date" content="2018-05-01" />
  <meta name="keywords" content="enhancers, transcription, transcription factor, epigenome" />
  <title>Total Functional Score of Enhancer Elements Identifies Lineage-Specific Enhancers that Drive Differentiation of Pancreatic Cells</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="github-pandoc.css">
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_CHTML-full" type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!-- Insert Analytics Script Below -->
  <script>
  </script>
  <!-- End Analytics Script -->
</head>
<body>
<header>
<h1 class="title">Total Functional Score of Enhancer Elements Identifies Lineage-Specific Enhancers that Drive Differentiation of Pancreatic Cells</h1>
</header>
<p><small><em> This manuscript (<a href="https://vsmalladi.github.io/tfsee-manuscript/v/d1ae40f4b1f15eb1604abd1f3d186ff103163fb7/">permalink</a>) was automatically generated from <a href="https://github.com/vsmalladi/tfsee-manuscript/tree/d1ae40f4b1f15eb1604abd1f3d186ff103163fb7">vsmalladi/tfsee-manuscript@d1ae40f</a> on May 1, 2018. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Venkat S. Malladi</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-0144-0564">0000-0002-0144-0564</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/vsmalladi">vsmalladi</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/katatonikkat">katatonikkat</a><br> <small> The Laboratory of Signaling and Gene Regulation, Cecil H. and Ida Green Center for Reproductive Biology Sciences and Division of Basic Reproductive Biology Research, Department of Obstetrics and Gynecology, University of Texas Southwestern Medical Center; Department of Bioinformatics, University of Texas Southwestern Medical Center&quot; </small></p></li>
<li><p><strong>Anusha Nagari</strong><br><br> <small> The Laboratory of Signaling and Gene Regulation, Cecil H. and Ida Green Center for Reproductive Biology Sciences and Division of Basic Reproductive Biology Research, Department of Obstetrics and Gynecology, University of Texas Southwestern Medical Center </small></p></li>
<li><p><strong>Hector L. Franco</strong><br><br> <small> The Laboratory of Signaling and Gene Regulation, Cecil H. and Ida Green Center for Reproductive Biology Sciences and Division of Basic Reproductive Biology Research, Department of Obstetrics and Gynecology, University of Texas Southwestern Medical Center; Department of Genetics and Lineberger Comprehensive Cancer Center, University of North Carolina at Chapel Hill&quot; </small></p></li>
<li><p><strong>W. Lee Kraus</strong><br><br> <small> The Laboratory of Signaling and Gene Regulation, Cecil H. and Ida Green Center for Reproductive Biology Sciences and Division of Basic Reproductive Biology Research, Department of Obstetrics and Gynecology, University of Texas Southwestern Medical Center </small></p></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>The ability to integrate different genome data sets to systematically identify active enhancers together with their cognate transcription factors (TF) remains a difficult and somewhat arbitratry process. We have developed a computational framework that systematically identifies active enhancers in any cell or tissue type together with the TFs bound at the enhancers by integrating multiple genomic assays that probe the transcriptional (GRO-seq and RNA-seq) and epigenetic (ChIP-seq) states of the cells. Our method, called Total Functional Score of Enhancer Elements (TFSEE), integrates the magnitude of enhancer transcription as a measure of enhancer activitity, enrichment of histone modifications typically associated with enhancers (H3K4me1 and H3K27ac), TF expression levels, and TF motif p-values to compute a probability score of TF binding events at active enhancers across the genome. This method has allowed us to define the enhancer landscape during differentiation of embryonic stem cells into pancreatic lineages and in breast cancer cells to define the regulatory pathways of the distinct molecular subtypes of breast cancer. Using TFSEE, we have identified key breast cancer subtype-specific transcription factors that are bound at active enhancers and dictate gene expression patterns determining growth outcomes. To demonstrate the broader utility of our approach, we have used this algorithm to identify transcription factors during the differentiation of embryonic stem cells into pancreatic cells. The analysis has revealed transcription factors maintaining the multipotency of endoderm stem cells and promoting differentiation into pancreatic progenitor cells. Taken together our results show that TFSEE can be used to perform multilayer genomic data integration to uncover novel cell type-specific transcription factors that control lineage-specific enhancers.</p>
<h2 id="background" class="page_break_before">Background</h2>
<p>Enhancers and the transcription factors (TFs) regulating their formation have been shown to play an important role in cell type-specific activation of gene expression <span class="citation" data-cites="sJDRq7sP d09ELIR1">[<a href="#ref-sJDRq7sP">1</a>,<a href="#ref-d09ELIR1">2</a>]</span>. Although thousands of potential enhancers have been identified in cell lines and tissues, identification of the enhancers that are active versus not active or poised remains a major challenge <span class="citation" data-cites="r7FNQqrP">[<a href="#ref-r7FNQqrP">3</a>]</span>. In addition, the ability to identify the TFs acting at the numerous enhancers in each cell type is technically challenging <span class="citation" data-cites="15J98V2qM sLkFMFZj">[<a href="#ref-15J98V2qM">4</a>,<a href="#ref-sLkFMFZj">5</a>]</span>.</p>
<p>Active enhancers have been shown to share several common features; such as increased chromatin accessibility (as measured by DNase-seq or ATAC-seq) <span class="citation" data-cites="NqDGzVRS x2PdIgDj 6xCXy2Jy">[<a href="#ref-NqDGzVRS">6</a>,<a href="#ref-x2PdIgDj">7</a>,<a href="#ref-6xCXy2Jy">8</a>]</span> and enrichment of post-translational modification of histone tails (as assessed by ChIP-seq), including H3K4me1 and H3K27ac <span class="citation" data-cites="vCijBy88 O5G8lPNO LkrR69b9">[<a href="#ref-O5G8lPNO">10</a>,<a href="#ref-LkrR69b9">11</a>,<a href="#ref-vCijBy88">9</a>]</span>. While these epigenetic features can successfully identify the location of many enhancers across the genome, they cannot readily differentiate between active and non-active enhancers <span class="citation" data-cites="1CNxOwAG6 UIPf9Y7q">[<a href="#ref-1CNxOwAG6">12</a>,<a href="#ref-UIPf9Y7q">13</a>]</span>. However, recent genomic assays have shown that enhancers tend to be bound by RNA polymerase II (Pol II) and transcribed, producing non-coding RNAs known as enhancer RNAs (‘eRNAs’) <span class="citation" data-cites="3ZN37fOe 5i6RU9Zx 1BkIYUDLC">[<a href="#ref-3ZN37fOe">14</a>,<a href="#ref-5i6RU9Zx">15</a>,<a href="#ref-1BkIYUDLC">16</a>]</span>. While the functions of the enhancer RNA transcripts are unknown, we and others have shown that enhancer transcription (as measured by total RNA-seq, GRO-seq or PRO-seq) can be used in the absence of any other genomic information to as a predictor of enhancer activity <span class="citation" data-cites="1BkIYUDLC 9nnd7Ryc 14GXIWfT0 1DE6cLccI 18olkFjWD lPKPHCBB d09ELIR1 GvDoUOfC 2eAh2YIr 5i6RU9Zx">[<a href="#ref-5i6RU9Zx">15</a>,<a href="#ref-1BkIYUDLC">16</a>,<a href="#ref-9nnd7Ryc">17</a>,<a href="#ref-14GXIWfT0">18</a>,<a href="#ref-1DE6cLccI">19</a>,<a href="#ref-d09ELIR1">2</a>,<a href="#ref-18olkFjWD">20</a>,<a href="#ref-lPKPHCBB">21</a>,<a href="#ref-GvDoUOfC">22</a>,<a href="#ref-2eAh2YIr">23</a>]</span>.</p>
<p>In recent years, advances in technology have facilitated the large scale functional characterization of enhancer activity <span class="citation" data-cites="14L6kLUE3 sKU362wY jjAPfyKA 5zmE7Qkb">[<a href="#ref-14L6kLUE3">24</a>,<a href="#ref-sKU362wY">25</a>,<a href="#ref-jjAPfyKA">26</a>,<a href="#ref-5zmE7Qkb">27</a>]</span> and annotation of genome-wide binding sites of TFs in various cell types and tissues <span class="citation" data-cites="15J98V2qM ZnmOeYfS">[<a href="#ref-ZnmOeYfS">28</a>,<a href="#ref-15J98V2qM">4</a>]</span>. However, due to countless cell types, experimental conditions and the large number of TFs <span class="citation" data-cites="EkABaVh5">[<a href="#ref-EkABaVh5">29</a>]</span>, integration of these independent data sets to achieve a comprehensive analysis of gene expression and actionable predictions of TFs driving cell type-specific gene expression can be very challenging. Furthermore, analyses that predict TF binding sites (TFBSs), which are usually 4-12 nucleotides in length <span class="citation" data-cites="uOXPLhWz">[<a href="#ref-uOXPLhWz">30</a>]</span>, using TF binding profile databases <span class="citation" data-cites="1AqKyPkB5 1DrZNoXOU 1C9AIanMe">[<a href="#ref-1AqKyPkB5">31</a>,<a href="#ref-1DrZNoXOU">32</a>,<a href="#ref-1C9AIanMe">33</a>]</span>, fail to consider that such sequences occur frequently by chance throughout the genome and that TF binding is cell type specific <span class="citation" data-cites="uuBDz4a3">[<a href="#ref-uuBDz4a3">34</a>]</span>. To overcome these limitations, we established a novel method called Total Functional Score of Enhancer Elements (TFSEE), which can be used to identify location and activity of enhancers in any cell or tissue type together with their cognate TFs.</p>
<p>In TFSEE, we integrate enhancer location and activity, TF motif prediction for each enhancer and the level of TF expression (Figure <a href="#fig:tfsee_processing">1</a>, <a href="#fig:overview_tfsee">2</a>). We have previously demonstrated TFSEE in the identification of key breast cancer subtype-specific transcription factors determining growth outcomes <span class="citation" data-cites="Iu6Tq2rd">[<a href="#ref-Iu6Tq2rd">35</a>]</span>. In the studies presented herein, we demonstrate the broader use of TFSEE to identify enhancers and TFs during the differentiation of embryonic stem cells into pancreatic progenitor cells. Taken together our results show that TFSEE can be used to perform multilayer genomic data integration to uncover novel cell type-specific transcription factors that control lineage-specific enhancers (Figure <a href="#fig:enhancer_predictions">3</a>A).</p>
<h2 id="results">Results</h2>
<h3 id="the-tfsee-model">The TFSEE model</h3>
<p>The TFSEE model integrates multiple genomics assays, GRO-seq, RNA-seq, and ChIP-seq, data with TF motif information to predict TFs driving the formation of active enhancers and the locations of their cognate enhancers. The TFSEE model consists of five key data processing steps (Figure <a href="#fig:tfsee_processing">1</a>) followed by a data integration stage (Figure <a href="#fig:overview_tfsee">2</a>). In step 1 of TFSEE, a universe of active enhancers across the different constituent cell types are identified. The enhancers can be identified either by enhancer transcription (GRO-seq or total RNA-seq) (Figure <a href="#fig:enhancer_pipeline">S1</a>A) or enrichment of epigenomic marks (H3K4me1 and H3K27ac) (Figure <a href="#fig:enhancer_pipeline">S1</a>B). In step 2 of TFSEE, genome-wide enhancer activity levels are assessed by calculating the enrichment (H3K4me1 and H3K27ac) and eRNA transcription (GRO-seq or total RNA-seq) profiles under the universe of enhancers per cell type. TFSEE was designed to detect enhancer activity changes and TF:enhancer links for each cell type. All TF to enhancer links are determined by a de novo motif search and summarizing the probability of that TF using the tools in steps 3-4 of TFSEE, which creates a table annotating enhancer to TF for each cell type. For all TFs identified TFSEE calculates the expression profile using (GRO-seq or RNA-seq) across every cell type in step 5.</p>
<p>The final stage integrates all the data in steps 1-5 (Figure <a href="#fig:overview_tfsee">2</a>) to determine TFSEE score matrix and heatmap. First, we generate an enhancer activity matrix A<sub>CxE</sub> for all cell types C for the universe active of E enhancers, as determined from step 2. We assume that the enhancer activity of each cell type is linearly correlated to the amount enhancer transcription (GRO-seq or total RNA-seq, G), and to the epigenomic marks (H3K4me1, M and H3K27ac, H). To reduce bias each individual enhancer enrichment is scaled between 0 and 1. Enhancer activity can be expressed as the following formula:</p>
<center>
<span class="math inline">\(A = G + M + H\)</span>
</center>
<p>Next, the enhancer activity matrix A<sub>CxE</sub>, is combined with motif prediction matrix T<sub>ExF</sub>, represent scaled motif prediction p-values, T, for each enhancer E, to form an intermediate matrix product. This matrix product is entrywise combined with TF expression matrix R, from step 5, the expression of each TF F for each cell type C, into a resulting matrix Z composed of C cell types and F TFs. TFSEE can be expressed as the following formula:</p>
<center>
<span class="math inline">\(Z =( A \times T) \circ R\)</span>
</center>
<h3 id="choice-of-biological-model-system-and-data">Choice of biological model system and data</h3>
<p>To better understand the TF-driven transcriptional programs using TFSEE, we used previously published transcriptional and epigenomic data from time course differentiation of human embryonic stem cells (hESC) towards pancreatic cell type <span class="citation" data-cites="BTqz3DNL 18a6G1TE5">[<a href="#ref-BTqz3DNL">36</a>,<a href="#ref-18a6G1TE5">37</a>]</span> (Figure <a href="#fig:enhancer_predictions">3</a>A). For these analyses, we used GRO-seq and RNA-seq, as well as ChIP-seq for 3 different histone modifications at five defined stages of differentiation: hESCs, definitive endoderm (DE), primitive gut tube (GT), posterior foregut (FG), and pancreatic endoderm (PE) (Figure <a href="#fig:enhancer_predictions">3</a>A, Table <a href="#tbl:data-sets">S1</a>). This embryonic development model allows us to explore the spatiotemporal gene regulation during development, by enhancers and TFs.</p>
<h3 id="unbiased-identification-of-enhancers-during-pancreatic-differentiation">Unbiased Identification of Enhancers during Pancreatic Differentiation</h3>
<p>We and other have shown that enhancers can be identified using enrichment of histone modifications (e.g. H3K4me1 and H3K27ac) <span class="citation" data-cites="vCijBy88 O5G8lPNO LkrR69b9">[<a href="#ref-O5G8lPNO">10</a>,<a href="#ref-LkrR69b9">11</a>,<a href="#ref-vCijBy88">9</a>]</span> or by enhancer transcription <span class="citation" data-cites="1BkIYUDLC">[<a href="#ref-1BkIYUDLC">16</a>]</span>. We used a computational pipeline to identify a universe of eRNA transcripts from GRO-seq (Figure <a href="#fig:enhancer_pipeline">S1</a>A) or enrichment of epigenomic marks (H3K4me1 or H3K27ac) (Figure <a href="#fig:enhancer_pipeline">S1</a>B) for the cell lines in the pancreatic differentiation time course model. All potential enhancers were filtered to be <span class="math inline">\(&gt;\)</span> 3 kb away from known transcription start sites (TSSs) of protein-coding genes from Gencode version 19 annotations <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">38</a>]</span>, and active promoters, as identified by H3K4me3 <span class="citation" data-cites="9O9ugXze">[<a href="#ref-9O9ugXze">39</a>]</span> (Figure <a href="#fig:density_plot">S2</a>A) to avoid complications in the analysis associated with overlapping promoter transcription.</p>
<p>Using enrichment of H3K4me1 and H3K27ac, RPKM cutoff of <span class="math inline">\(\geq\)</span> 1 (Figure <a href="#fig:density_plot">S2</a>B and C) in at least one cell line, we determined there to be set of 182,335 candidate enhancers across all stages of pancreatic differentiation (Figure <a href="#fig:enhancer_predictions">3</a>B). We categorized these candidate enhancers for each cell line and found that <span class="math inline">\(\le\)</span> 20% of the enhancers, as determined by presences of H3K4me1 and H3K27ac, are active in each cell line and the majority are marked by only H3K4me1 (Figure <a href="#fig:enhancer_predictions">3</a>B). These results confirm the enhancer landscape across pancreatic differentiation reported by Wang <em>et al</em>. We then identified a set of 4,974 candidate enhancers (Figure <a href="#fig:enhancer_predictions">3</a>B) by GRO-seq as described previously <span class="citation" data-cites="1DE6cLccI">[<a href="#ref-1DE6cLccI">19</a>]</span>, using RPKM <span class="math inline">\(\geq\)</span> 0.5 or <span class="math inline">\(\geq\)</span> 1 (Figure <a href="#fig:density_plot">S2</a>D and E) in at least one cell line. Compared to active enhancer by histone modifications, we found that the number of active enhancers in each cell line ranged from 77-25% of all candidate enhancers.</p>
<p>We compared the overlap from histone enhancer prediction methods (H3K4me1 or H3K27ac) to output from an enhancer transcription based approach (GRO-seq). We found that 12% of enhancers called based on enhancer transcription using GRO-seq data are identified by all the of the other methods (enrichment of H3K4me1 and H3K27ac) (Figure <a href="#fig:enhancer_predictions">3</a>C, <a href="#fig:genome_browser">S3</a>A). In contrast, greater than 75% of the enhancers were solely identified by enhancer transcription (Figure <a href="#fig:enhancer_predictions">3</a>C and D, <a href="#fig:genome_browser">S3</a>B). Although H3K27ac and H3K4me1 might be two histone modifications commonly associated with enhancers, these are not the only chromatin mark involved and other modifications may be present that were not assayed for <span class="citation" data-cites="crCcep4D">[<a href="#ref-crCcep4D">40</a>]</span>. Additionally, less than 1% of enhancers called based on enrichment of H3K4me1 or H3K27ac are identified by the other methods (Figure <a href="#fig:enhancer_predictions">3</a>C). This may be due, in part, to the fact that enhancer calling based on H3K4me1 or H3K27ac enrichment, yields much larger numbers of putative enhancers (Figure <a href="#fig:enhancer_predictions">3</a>D), many of which may be false positives or inactive as the true regulatory elements (Figure <a href="#fig:genome_browser">S3</a>C and D). Nonetheless, as we show below, incorporating enhancer transcription into an TFSEE pipeline that includes information about H3K4me1 and H3K27ac enrichment, improves the identification cell type-specific enhancers.</p>
<h3 id="tfsee-identifies-cell-type-specific-enhancers-and-their-cognate-tfs">TFSEE identifies cell type-specific enhancers and their cognate TFs</h3>
<p>We used the enhancer calls by Figure <a href="#fig:enhancer_predictions">3</a>B, to identify cell type-specific enhancers and their cognate TFs, using TFSEE, either by enhancer transcription or enrichment of epigenomic marks. We visualized the results from TFSEE using unsupervised hierarchical clustering, which grouped the cell types into two major clades: (1) FG, and PE (2) hESC, DE, and GT (Figure <a href="#fig:tfsee_groseq">4</a>A, <a href="#fig:tfsee_histone">S4</a>A). To better understand the TF:enhancer dynamics across all differentiation stages we clustered the TSEE score across all differentiation stages, revealing four major categories using enhancer transcription (Figure <a href="#fig:tfsee_groseq">4</a>B). We examined the enrichment of putative enhancers and their associated TFs across stages by quantifying their normalized TFSEE score. This analysis revealed four major clusters: 1. driving early (hESC, DE) and late pancreatic differentiation (FG and PE), 2. enriched in GT, 3. driving pre-pancreatic linage (hESC, DE and GT), and 4. driving late-pancreatic differentiation (FG and PE) (Figure <a href="#fig:tfsee_groseq">4</a>D). In contrast, using only histone enrichment to identify enhancers, we retrieve only three clusters (Figure <a href="#fig:tfsee_histone">S4</a>B). These results highlight TF:enhancers driving pre-pancreatic lineage (hESC, DE and GT), and late-pancreatic differentiation (FG and PE), but fails to highlight any other stage specific drivers (Figure <a href="#fig:tfsee_histone">S4</a>C). We were particularly interested in TFs and enhancers that provided a clear demarcation of enrichment between pre- and late- pancreatic differentiation.</p>
<p>To investigate the the distinct roles of lineage specific enhancers and their cognate TFs, we first examined the mRNA levels of the corresponding predicted TFs of each cluster in each of the stages. Our analysis revealed that TFs identified in pre-pancreatic lineage show equal expression across stages, while late-pancreatic TFs are highly expressed in FG and PE (Figure <a href="#fig:late_pre_diff">5</a>A, <a href="#fig:late_pre_histone">S5</a>A) coinciding with pancreatic induction at the FG stage (Figure <a href="#fig:enhancer_predictions">3</a>A). Conversely, we didn’t see an enrichment of TFs in a stage specific manner for either TFs enriched early (hESC, DE) and late pancreatic differentiation (FG and PE) or those maintaining GT pluripotency (Figure <a href="#fig:primitive_gut">S6</a>A).</p>
<p>Next, we determined if enhancer transcription corresponding to the enriched TFs, using TFSEE score, and the regulation of their nearby genes might regulate differentiation biology. To do so, we identified the enhancers corresponding to the predicted TFs using enriched binding motif prediction, and then determined the level of transcription for each enhancer, using GRO-seq or H3K27ac ChIP-seq, (Figure <a href="#fig:late_pre_diff">5</a>B, <a href="#fig:late_pre_histone">S5</a>B, <a href="#fig:primitive_gut">S6</a>B) and the nearest neighboring gene (upstream or downstream), using RNA-seq (Figure <a href="#fig:late_pre_diff">5</a>C, <a href="#fig:late_pre_histone">S5</a>C, <a href="#fig:primitive_gut">S6</a>C). Interestingly, transcribed enhancers exhibited stage specific enrichment, which doesn’t correspond to the patterns found from TFSEE enrichment (Figure <a href="#fig:late_pre_diff">5</a>B, <a href="#fig:late_pre_histone">S5</a>B, <a href="#fig:primitive_gut">S6</a>B). This result reflects that 48% - 99% of the enhancers are shared between clusters and the variation between clusters is due to differences in TF expression and affinity to motifs. Likewise, the nearest neighboring gene for each transcribed enhancer doesn’t exhibit stage specific enrichment (Figure <a href="#fig:late_pre_diff">5</a>C, <a href="#fig:late_pre_histone">S5</a>C, <a href="#fig:primitive_gut">S6</a>C) due to the vast abundance of enhancers and thus neighboring genes shared between the clusters. However, without further high-throughput data to study promoter-enhancer linking (as measured by 4C, ChIA-PET, or Hi-C) <span class="citation" data-cites="zQrKrStO">[<a href="#ref-zQrKrStO">41</a>]</span> it is difficult to understand the stage specific regulatory network.</p>
<p>To further understand the potential regulators of each cluster we determined a rank order frequency distribution for all TFs within each cluster (Figure <a href="#fig:late_pre_diff">5</a>D and E). This analysis revealed enrichment of HINFP, RARG, ZIC3, and SP1-like family TFs (SP1 and SP8) which are important regulators of embryonic development <span class="citation" data-cites="gM7zvZej">[<a href="#ref-gM7zvZej">44</a>]</span> (Figure <a href="#fig:late_pre_diff">5</a>D). Additionally, the Onecut family (ONECUT2 and ONECUT3), EGR1, MITF and FOXP1 TFs are enriched in cluster 4 and have been shown to function in pancreatic and islet cell development <span class="citation" data-cites="1A0K4olzK">[<a href="#ref-1A0K4olzK">48</a>]</span> (Figure <a href="#fig:late_pre_diff">5</a>E). Similar results were obtained for comparison of TFSEE scores from different clusters using GRO-seq and Histone called enhancers (Figure <a href="#fig:primitive_gut">S6</a>D and E, <a href="#fig:late_pre_histone">S5</a>D and E).</p>
<p>To compare TFs enriched between GRO-seq and Histone driven TFSEE scores we compared the TFs enriched in pre- and late-pancreatic differentiation. We found 9 and 12 TFs enriched in common for pre- and late-pancreatic respectively. The differences in TFs that are enriched may be due, in part, to enhancer calling using H3K4me1 and H3K27ac enrichment, yields much larger numbers of putative enhancers, many of which may be false positives or inactive as true regulatory elements producing more varied enriched TF motifs. Altogether, our results show that TFSEE can be used to identify cell type specific TFs that control the biology of those cell types.</p>
<h2 id="discussion">Discussion</h2>
<h2 id="acknowledgments">Acknowledgments</h2>
<h2 id="material-and-methods" class="page_break_before">Material and Methods</h2>
<h3 id="genomic-data-curation">Genomic Data Curation</h3>
<p>We used previously published GRO-seq, ChIP-seq and RNA-seq data from <span class="citation" data-cites="BTqz3DNL 18a6G1TE5">[<a href="#ref-BTqz3DNL">36</a>,<a href="#ref-18a6G1TE5">37</a>]</span> time course differentiation of human embryonic stem cells (hESC) to pancreatic endoderm (PE). All data sets are available from NCBI’s Gene Expression Omnibus <span class="citation" data-cites="Bc3QGVe7">[<a href="#ref-Bc3QGVe7">52</a>]</span> or EMBL-EBI’s ArrayExpress <span class="citation" data-cites="937jwJMM">[<a href="#ref-937jwJMM">53</a>]</span> repositories using the accession numbers listed in Table <a href="#tbl:data-sets">S1</a>.</p>
<a name="tbl:data-sets"></a>
<table>
<caption><span>Table S1:</span> <strong>Description and accession numbers of GRO-seq, ChIP-seq and RNA-seq datasets.</strong> </caption>
<colgroup>
<col style="width: 26%" />
<col style="width: 73%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Assay</th>
<th style="text-align: center;">Accessions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">GRO-seq</td>
<td style="text-align: center;">GSM1316306, GSM1316313, GSM1316320, GSM1316327, GSM1316334</td>
</tr>
<tr class="even">
<td style="text-align: center;">H3K4me3 ChIP-seq</td>
<td style="text-align: center;">ERR208008, ERR208014, ERR207998, ERR20798, ERR207999</td>
</tr>
<tr class="odd">
<td style="text-align: center;">H3K4me1 ChIP-seq</td>
<td style="text-align: center;">GSM1316302, GSM1316303, GSM1316309, GSM1316316, GSM1316317, GSM1316310, GSM1316323, GSM1316324, GSM1316330, GSM1316331</td>
</tr>
<tr class="even">
<td style="text-align: center;">H3K27ac ChIP-seq</td>
<td style="text-align: center;">GSM1316300, GSM1316301, GSM1316307, GSM1316308, GSM1316314, GSM1316315, GSM1316321, GSM1316322, GSM1316328, GSM1316329</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Input ChIP-seq</td>
<td style="text-align: center;">ERR208001, ERR208012, ERR207984, ERR208011, ERR207986, GSM1316304, GSM1316305, GSM1316311, GSM1316312, GSM1316318, GSM1316319, GSM1316325, GSM1316326, GSM1316332, GSM1316333</td>
</tr>
<tr class="even">
<td style="text-align: center;">RNA-seq</td>
<td style="text-align: center;">ERR266333, ERR266335, ERR266337, ERR266338, ERR266341, ERR266342, ERR266344, ERR266346, ERR266349, ERR266351</td>
</tr>
</tbody>
</table>
<h3 id="analysis-of-chip-seq-data-sets">Analysis of ChIP-seq Data Sets</h3>
<p>The raw reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in Bowtie version 1.0.0 <span class="citation" data-cites="RU8ikjzP">[<a href="#ref-RU8ikjzP">54</a>]</span>. The aligned reads were subsequently filtered for quality and uniquely mappable reads were retained for further analysis using Samtools version 0.1.19 <span class="citation" data-cites="IPc8cy7s">[<a href="#ref-IPc8cy7s">55</a>]</span> and Picard version 1.127 <span class="citation" data-cites="tfF98ztu">[<a href="#ref-tfF98ztu">56</a>]</span>. Library complexity was measured using BEDTools version 2.17.0 <span class="citation" data-cites="1HWiAHnIw">[<a href="#ref-1HWiAHnIw">57</a>]</span> and meets ENCODE data quality standards <span class="citation" data-cites="hvhWxir9">[<a href="#ref-hvhWxir9">58</a>]</span>. Relaxed peaks were called using MACS version 2.1.0 <span class="citation" data-cites="Bhecn2fS">[<a href="#ref-Bhecn2fS">59</a>]</span> with a p-value of <span class="math inline">\(1 \times 10^{-2}\)</span> for each replicate, pooled replicates’ reads and pseudoreplicates. Peak calls from the pooled replicates that are either observed in both replicates, or in both pseudoreplicates were used for subsequent analysis.</p>
<h3 id="analysis-of-rna-seq-data-sets">Analysis of RNA-seq Data Sets</h3>
<p>The raw reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in STAR version 2.4.2a <span class="citation" data-cites="tTu8Ds9Z">[<a href="#ref-tTu8Ds9Z">60</a>]</span>. Quantification of genes against Gencode version 19 <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">38</a>]</span> annotations was done using default parameters in RSEM version 1.2.31 <span class="citation" data-cites="Dh2n1EV3">[<a href="#ref-Dh2n1EV3">61</a>]</span>.</p>
<h3 id="analysis-of-gro-seq-data">Analysis of GRO-seq Data</h3>
<p>The GRO-seq reads were trimmed to the first 36 bases to trim adapter and low quality sequence, using default parameters of fastx_trimer in fastx-toolkit version 0.0.13.2 <span class="citation" data-cites="jwUQ55cX">[<a href="#ref-jwUQ55cX">62</a>]</span>. The trimmed reads were aligned to the human reference genome (GRCh37/hg19) using default parameters in BWA version 0.7.12 <span class="citation" data-cites="roi5bwXL">[<a href="#ref-roi5bwXL">63</a>]</span>.</p>
<h3 id="kernel-density">Kernel Density</h3>
<p>Kernel density plot representations were used to express the univariate distribution of ChIP-seq reads under peaks, RNA-seq reads for protein-coding genes and GRO-seq reads for short paired and short unpaired eRNAs. The kernel density plots were calculated in Python (ver. 2.7.11) using the kdeplot function from <a href="http://seaborn.pydata.org/">seaborn</a> version 0.7.1 <span class="citation" data-cites="qxfTM222">[<a href="#ref-qxfTM222">64</a>]</span> with default parameters.</p>
<h3 id="defining-transcription-start-sites-and-promoters">Defining Transcription Start Sites and Promoters</h3>
<p>We made distinct transcription start sites (TSS) for protein-coding genes from Gencode version 19 <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">38</a>]</span> annotations using MakeGencodeTSS <span class="citation" data-cites="tDy18Yol">[<a href="#ref-tDy18Yol">65</a>]</span>. We identified active promoters, as identified by H3K4me3 <span class="citation" data-cites="9O9ugXze">[<a href="#ref-9O9ugXze">39</a>]</span>. A RPKM cutoff of <span class="math inline">\(\geq\)</span> 1 for H3K4me3 in at least one cell line was used to identify a peak as an active enhancer (Figure <a href="#fig:density_plot">S2</a>A).</p>
<h3 id="enhancer-calling-by-chip-seq">Enhancer calling by ChIP-seq</h3>
<h4 id="calling-active-enhancers."><em>Calling Active Enhancers</em>.</h4>
<p>We built a universe of peak calls by merging the peaks from individual cell lines for histone modifications (H3K4me1 and H3K27ac) and stratifying the boundaries to remove overlaps/redundancies occurring from the union of all peaks. Potential enhancers were defined as peaks that were <span class="math inline">\(&gt;\)</span> 3kb from known TSS, protein coding genes from Gencode version 19 annotations <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">38</a>]</span>, and H3K4me3 peaks. A RPKM cutoff of <span class="math inline">\(\geq\)</span> 1 for H3K4me1 and H3K27ac (Figure <a href="#fig:density_plot">S2</a>B and C) in at least one cell line was used to identify a peak as an active enhancer. The universe of active enhancers was assembled using the cutoffs noted above for each cell line and was used for further analyses.</p>
<h4 id="motif-analyses."><em>Motif Analyses</em>.</h4>
<p>De novo motif analyses were performed on a 1 kb region (<span class="math inline">\(\pm\)</span> 500 bp) surrounding the peak summit for the top 10000 enhancers, using the command-line version of MEME-ChIP from MEME Suite version 4.11.1 <span class="citation" data-cites="dwj6qSn3 TfukIx2u">[<a href="#ref-dwj6qSn3">66</a>,<a href="#ref-TfukIx2u">67</a>]</span>. The following parameters were used for motif prediction: (1) zero or one occurrence per sequence (-mod zoops); (2) number of motifs (-nmotifs 15); (3) minimum, maximum width of the motif (-minw 8, -maxw 15). All the other parameters were set at the default. The predicted motifs from MEME were matched to known motifs in the JASPAR database (JASPAR_CORE_2016_vertebrates.meme) <span class="citation" data-cites="1DrZNoXOU">[<a href="#ref-1DrZNoXOU">32</a>]</span> using TOMTOM <span class="citation" data-cites="1C9AIanMe">[<a href="#ref-1C9AIanMe">33</a>]</span>.</p>
<h3 id="enhancer-calling-by-gro-seq">Enhancer calling by GRO-seq</h3>
<h4 id="transcript-calling."><em>Transcript calling</em>.</h4>
<p>Transcript calling was performed using a two-state hidden Markov model using the groHMM data analysis package version 3.4 <span class="citation" data-cites="1BkIYUDLC 1HVy2rbI lPKPHCBB">[<a href="#ref-1BkIYUDLC">16</a>,<a href="#ref-lPKPHCBB">21</a>,<a href="#ref-1HVy2rbI">68</a>]</span> on each individual cell lines. The negative log transition probability of the switch between transcribed state to non-transcribed state and the variance in read counts in the non-transcribed state that are used to predict the transcription units for the cell lines in this study are listed Table <a href="#tbl:groseq-tune">S2</a>.</p>
<a name="tbl:groseq-tune"></a>
<table>
<caption><span>Table S2:</span> <strong>groHMM tunning parameters.</strong> </caption>
<thead>
<tr class="header">
<th style="text-align: center;">Cell Line</th>
<th style="text-align: center;">-Log Transition Probability</th>
<th style="text-align: center;">Variance in read counts</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">hES</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">45</td>
</tr>
<tr class="even">
<td style="text-align: center;">DE</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">35</td>
</tr>
<tr class="odd">
<td style="text-align: center;">GT</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">50</td>
</tr>
<tr class="even">
<td style="text-align: center;">FG</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">35</td>
</tr>
<tr class="odd">
<td style="text-align: center;">PE</td>
<td style="text-align: center;">50</td>
<td style="text-align: center;">35</td>
</tr>
</tbody>
</table>
<p>We then built a universe of transcripts by merging the groHMM-called transcripts from individual cell lines and stratifying the boundaries to remove overlaps/redundancies occurring from the union of all transcripts.</p>
<h4 id="calling-enhancer-transcripts."><em>Calling Enhancer Transcripts</em>.</h4>
<p>We filtered and collected a subset of short intergenic transcripts <span class="math inline">\(&lt;\)</span> 9 kb in length and <span class="math inline">\(&gt;\)</span> 3 kb away from known transcription start sites (TSSs) of protein-coding genes from Gencode version 19 annotations <span class="citation" data-cites="15rkMH6SD">[<a href="#ref-15rkMH6SD">38</a>]</span>, and H3K4me3 peaks. These were further classified into (1) short paired eRNAs and (2) short unpaired eRNAs as described previously <span class="citation" data-cites="1DE6cLccI">[<a href="#ref-1DE6cLccI">19</a>]</span>. For the short paired eRNAs, the sum of the GRO-seq RPKM values for both strands of DNA was used to determine if an enhancer transcript pair is expressed using a cutoff of RPKM <span class="math inline">\(\geq\)</span> 0.5 (Figure <a href="#fig:density_plot">S2</a>D). An RPKM cutoff of <span class="math inline">\(\geq\)</span> 1 was used to determine the universe expressed short unpaired eRNAs (Figure <a href="#fig:density_plot">S2</a>E). The comprehensive of expressed eRNAs (short paired and short unpaired) was assembled using the cutoffs noted above for each cell line was used for further analyses.</p>
<h4 id="motif-analyses.-1"><em>Motif Analyses</em>.</h4>
<p>De novo motif analyses was performed on a 1 kb region (<span class="math inline">\(\pm\)</span> 500 bp) surrounding the overlap center or the transcription start site for short paired and short unpaired eRNAs, respectively, using the command-line version of MEME from MEME Suite version 4.11.1 <span class="citation" data-cites="dwj6qSn3">[<a href="#ref-dwj6qSn3">66</a>]</span>. The following parameters were used for motif prediction: (1) zero or one occurrence per sequence (-mod zoops); (2) number of motifs (-nmotifs 15); (3) minimum, maximum width of the motif (-minw 8, -maxw 15); and (4) search for motif in given strand and reverse complement strand (-revcomp). The predicted motifs from MEME were matched to known motifs in the JASPAR database (JASPAR_CORE_2016_vertebrates.meme) <span class="citation" data-cites="1DrZNoXOU">[<a href="#ref-1DrZNoXOU">32</a>]</span> using TOMTOM <span class="citation" data-cites="1C9AIanMe">[<a href="#ref-1C9AIanMe">33</a>]</span>.</p>
<h3 id="generating-heatmaps-and-clusters">Generating Heatmaps and Clusters</h3>
<p>For each cell line, the functional scores were Z-score normalized. To identify cognate transcription factors by cell type, we performed hierarchical clustering by calculating the Euclidean distance using clustermap from <a href="http://seaborn.pydata.org/">seaborn</a> version 0.7.1 <span class="citation" data-cites="qxfTM222">[<a href="#ref-qxfTM222">64</a>]</span>. For visualization of the multidimensional TFSEE scores, we performed t-distributed stochastic neighbor embedding analysis (t-SNE) <span class="citation" data-cites="N7ngUHQX">[<a href="#ref-N7ngUHQX">69</a>]</span> using the TSNE function and labeled the clusters by calculating K-means clustering using the KMeans function with the expectation-maximization algorithm in <a href="http://scikit-learn.org/">scikit-learn</a> version 0.17.1 <span class="citation" data-cites="vHIKEjNs GGWEfSeb r8BJ7tJV">[<a href="#ref-vHIKEjNs">70</a>,<a href="#ref-GGWEfSeb">71</a>,<a href="#ref-r8BJ7tJV">72</a>]</span>.</p>
<h3 id="nearest-neighboring-gene-analyses-and-box-plots">Nearest Neighboring Gene Analyses and Box Plots</h3>
<p>The universe of expressed genes in each cell line was determined from the RNA-seq data using a FPKM cutoff of <span class="math inline">\(&gt;\)</span> 0.4 (Figure <a href="#fig:density_plot">S2</a>F). The set of nearest neighboring expressed genes for each enhancer defined by an expressed eRNA or the enrichment of active histone marks was determined for each cell line. Box plot representations were used to express the levels of transcription or enrichment for each called enhancer and transcription of their nearest neighboring expressed genes. The read distribution (RPKM) for each enhancer or (FPKM) gene was calculated and plotted using the boxplot function from <a href="https://matplotlib.org/">matplotlib</a> version 2.0.2 <span class="citation" data-cites="1026Gxdsi oaruRtzO">[<a href="#ref-1026Gxdsi">73</a>,<a href="#ref-oaruRtzO">74</a>]</span>. Wilcoxon rank sum tests were performed to determine the statistical significance of all comparisons.</p>
<h2 id="figures-and-figure-legends" class="page_break_before">Figures and Figure Legends</h2>
<figure>
<img src="images/tfsee_processing.png" alt="Figure 1: Data Processing for Total Functional Score of Enhancer Elements (TFSEE) Method. The TFSEE method has five data processing steps that are used to identify enhancer location and activity and their cognate transcription factors (TFs). In step 1, epigenomic (ChIP-seq) or the transcriptional (GRO-seq or total RNA-seq) profiles are used to generate a universe of active enhancers across the different constituent cell types. In step 2, TFSEE calculates the enrichment (H3K4me1 and H3K27ac) and eRNA transcription (GRO-seq and total RNA-seq) profiles under all identified active enhancers per cell type. Cell type-specific enhancers are used as input for step 3, where a de novo motif search is performed to identify potential TFs at each enhancer. If a motif is represented multiple times for a given enhancer location, TFSEE combines the probability of that motif into a single p-value in step 4. Step 5 integrates the amount of eRNA transcription (GRO-seq or total RNA-seq) and the expression of the TFs whose motifs were predicted in step 3 and 4 for all cell types, to provide an output of TF expression profiles across every cell type." id="fig:tfsee_processing" /><figcaption><span>Figure 1:</span> <strong>Data Processing for Total Functional Score of Enhancer Elements (TFSEE) Method.</strong> The TFSEE method has five data processing steps that are used to identify enhancer location and activity and their cognate transcription factors (TFs). In step 1, epigenomic (ChIP-seq) or the transcriptional (GRO-seq or total RNA-seq) profiles are used to generate a universe of active enhancers across the different constituent cell types. In step 2, TFSEE calculates the enrichment (H3K4me1 and H3K27ac) and eRNA transcription (GRO-seq and total RNA-seq) profiles under all identified active enhancers per cell type. Cell type-specific enhancers are used as input for step 3, where a de novo motif search is performed to identify potential TFs at each enhancer. If a motif is represented multiple times for a given enhancer location, TFSEE combines the probability of that motif into a single p-value in step 4. Step 5 integrates the amount of eRNA transcription (GRO-seq or total RNA-seq) and the expression of the TFs whose motifs were predicted in step 3 and 4 for all cell types, to provide an output of TF expression profiles across every cell type.</figcaption>
</figure>
<figure>
<img src="images/overview_tfsee.png" alt="Figure 2: Overview of Total Functional Score of Enhancer Elements (TFSEE) Method. TFSEE combines diverse data sets to identify enhancer location and activity and their cognate transcription factors (TFs). An illustration of TFSEE data integration stage, taking the outputs generated in panel A, to identify the location, activity level, and predicted TFs at each enhancer across all cell types. (Top) All matrices represent scaled enhancer activity for each cell type in each enhancer prediction method (G, H, and M). All matrices are linearly combined into a resulting matrix A, to provide a total enhancer activity score. (Bottom) Enhancer activity matrix A, is combined with motif prediction matrix T, represent scaled motif prediction p-values for each enhancer, to form an intermediate matrix product. This matrix product is entrywise combined with TF expression matrix R (scaled TF expression for each cell type), into a resulting matrix Z, on which TFSEE clustering is performed." id="fig:overview_tfsee" /><figcaption><span>Figure 2:</span> <strong>Overview of Total Functional Score of Enhancer Elements (TFSEE) Method.</strong> TFSEE combines diverse data sets to identify enhancer location and activity and their cognate transcription factors (TFs). An illustration of TFSEE data integration stage, taking the outputs generated in panel A, to identify the location, activity level, and predicted TFs at each enhancer across all cell types. (Top) All matrices represent scaled enhancer activity for each cell type in each enhancer prediction method (G, H, and M). All matrices are linearly combined into a resulting matrix A, to provide a total enhancer activity score. (Bottom) Enhancer activity matrix A, is combined with motif prediction matrix T, represent scaled motif prediction p-values for each enhancer, to form an intermediate matrix product. This matrix product is entrywise combined with TF expression matrix R (scaled TF expression for each cell type), into a resulting matrix Z, on which TFSEE clustering is performed.</figcaption>
</figure>
<figure>
<img src="images/enhancer_predictions.png" alt="Figure 3: Comparison of genome-wide prediction of enhancers during pancreatic differentiation. (A) (Top) Schematic of pancreatic differentiation starting from Human embryonic stem cells (hESCs) to pancreatic endoderm (PE). (Bottom) Depiction of epigenomic (ChIP-seq) and transcriptional (GRO-seq and RNA-seq) profiles for each cell line used for analysis. (B) Stacked bar chart comparing expression of candidate enhancers categorized by (Top) H3K4me1 and H3K27ac enrichment, or (Bottom) enhancer transcription (GRO-seq). (C) Stacked bar chart comparing enhancer prediction methods in pancreatic differentiation. Enhancers were called using enhancer transcription (GRO-seq) or by using H3K4me1 enrichment, or H3K27ac enrichment. The percentage of called enhancers from one prediction method that overlap with enhancers called using other methods is shown. (D) UpSet plot showing the set intersection of enhancer identification methods shown in panel C." id="fig:enhancer_predictions" /><figcaption><span>Figure 3:</span> <strong>Comparison of genome-wide prediction of enhancers during pancreatic differentiation.</strong> <strong>(A)</strong> <em>(Top)</em> Schematic of pancreatic differentiation starting from Human embryonic stem cells (hESCs) to pancreatic endoderm (PE). <em>(Bottom)</em> Depiction of epigenomic (ChIP-seq) and transcriptional (GRO-seq and RNA-seq) profiles for each cell line used for analysis. <strong>(B)</strong> Stacked bar chart comparing expression of candidate enhancers categorized by <em>(Top)</em> H3K4me1 and H3K27ac enrichment, or <em>(Bottom)</em> enhancer transcription (GRO-seq). <strong>(C)</strong> Stacked bar chart comparing enhancer prediction methods in pancreatic differentiation. Enhancers were called using enhancer transcription (GRO-seq) or by using H3K4me1 enrichment, or H3K27ac enrichment. The percentage of called enhancers from one prediction method that overlap with enhancers called using other methods is shown. <strong>(D)</strong> UpSet plot showing the set intersection of enhancer identification methods shown in panel C.</figcaption>
</figure>
<figure>
<img src="images/tfsee_groseq.png" alt="Figure 4: TFSEE identifies cell type-specific enhancers and their cognate TFs that drive gene expression during pancreatic differentiation. (A) Unsupervised hierarchical clustering of cell type-normalized TFSEE scores shown in a heatmap representation. hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). (B) Biaxial t-SNE clustering plot of cell type-normalized TFSEE scores showing evidence of four distinct clusters, each point represents an individual TF. (C) Box plots of normalized TFSEE score for clusters identified in pancreatic differentiation (panel B), number of TFs are indicated in each cluster. Bars marked with different letters are significantly different (Wilcoxon rank sum test, p \lt 1 \times 10^{-4}). Cluster 1, TFs associated with early (hESC, DE) and late pancreatic differentiation (FG and PE). Cluster 2, TFs associated with GT pluripotency. Cluster 3, TFs associated with pre-pancreatic lineage induction (hESC, DE and GT). Cluster 4, TFs associated with late-pancreatic differentiation (FG and PE)." id="fig:tfsee_groseq" /><figcaption><span>Figure 4:</span> <strong>TFSEE identifies cell type-specific enhancers and their cognate TFs that drive gene expression during pancreatic differentiation.</strong> <strong>(A)</strong> Unsupervised hierarchical clustering of cell type-normalized TFSEE scores shown in a heatmap representation. hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). <strong>(B)</strong> Biaxial t-SNE clustering plot of cell type-normalized TFSEE scores showing evidence of four distinct clusters, each point represents an individual TF. <strong>(C)</strong> Box plots of normalized TFSEE score for clusters identified in pancreatic differentiation (panel B), number of TFs are indicated in each cluster. Bars marked with different letters are significantly different (Wilcoxon rank sum test, <span class="math inline">\(p \lt 1 \times 10^{-4}\)</span>). Cluster 1, TFs associated with early (hESC, DE) and late pancreatic differentiation (FG and PE). Cluster 2, TFs associated with GT pluripotency. Cluster 3, TFs associated with pre-pancreatic lineage induction (hESC, DE and GT). Cluster 4, TFs associated with late-pancreatic differentiation (FG and PE).</figcaption>
</figure>
<figure>
<img src="images/late_pre_diff.png" alt="Figure 5: TFSEE-Predicted TFs are enriched in pre- and late- pancreatic differentiation. (A-C) Box plots of normalized TF expression (panel A), enhancer transcription (panel B), and gene expression for the nearest neighboring genes to active enhancers (panel C) in pre- (cluster 3) and late-pancreatic (cluster 4) differentiation across the different cell types. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test). hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). (A) TFs identified in cluster 3 by TFSEE show equal expression across differentiation. While, cluster 4 highlights TFs highly expressed in FG and PE. TF expression as measured by RNA-seq. Number of TFs in each cluster are in parenthesis. (p \lt 1 \times 10^{-4}) (B) Enhancer transcription as measured by GRO-seq. Number of enhancers in each cluster are in parenthesis. p \lt 1 \times 10^{-4}). (C) Gene expression as measured by RNA-seq. Number of genes in each cluster are in parenthesis. (p \lt 0.05) (D and E) Rank order of TFs enriched in the Cluster 3 and the Cluster 4 identified using TFSEE. The top ten TFs in each Cluster are noted." id="fig:late_pre_diff" /><figcaption><span>Figure 5:</span> <strong>TFSEE-Predicted TFs are enriched in pre- and late- pancreatic differentiation.</strong> <strong>(A-C)</strong> Box plots of normalized TF expression (panel A), enhancer transcription (panel B), and gene expression for the nearest neighboring genes to active enhancers (panel C) in pre- (cluster 3) and late-pancreatic (cluster 4) differentiation across the different cell types. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test). hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). <strong>(A)</strong> TFs identified in cluster 3 by TFSEE show equal expression across differentiation. While, cluster 4 highlights TFs highly expressed in FG and PE. TF expression as measured by RNA-seq. Number of TFs in each cluster are in parenthesis. (<span class="math inline">\(p \lt 1 \times 10^{-4}\)</span>) <strong>(B)</strong> Enhancer transcription as measured by GRO-seq. Number of enhancers in each cluster are in parenthesis. <span class="math inline">\(p \lt 1 \times 10^{-4}\)</span>). <strong>(C)</strong> Gene expression as measured by RNA-seq. Number of genes in each cluster are in parenthesis. (<span class="math inline">\(p \lt 0.05\)</span>) <strong>(D and E)</strong> Rank order of TFs enriched in the Cluster 3 and the Cluster 4 identified using TFSEE. The top ten TFs in each Cluster are noted.</figcaption>
</figure>
<figure>
<img src="images/enhancer_pipeline.png" alt="Figure S1: Unbiased, genome-wide prediction of active enhancers. (A) Overview of the computational pipeline used for the genome-wide annotation of enhancer transcripts (eRNAs) and prediction of active enhancers using GRO-seq data. (B) Overview of the computational pipeline used for the genome-wide annotation of and prediction of active enhancers using ChIP-seq (H3K4me1 and H3K27ac) data." id="fig:enhancer_pipeline" data-tag="S1" /><figcaption><span>Figure S1:</span> <strong>Unbiased, genome-wide prediction of active enhancers.</strong> <strong>(A)</strong> Overview of the computational pipeline used for the genome-wide annotation of enhancer transcripts (eRNAs) and prediction of active enhancers using GRO-seq data. <strong>(B)</strong> Overview of the computational pipeline used for the genome-wide annotation of and prediction of active enhancers using ChIP-seq (H3K4me1 and H3K27ac) data.</figcaption>
</figure>
<figure>
<img src="images/density_plot.png" alt="Figure S2: Density plots of enhancer and gene expression levels across all cell types. Kernel density plots of log-transformed RPKM and FPKM values for determining active enhancers and genes. The dashed grey line represents the minimum expression cutoff. (A) Density plot of H3K4me3 (promoter mark) cutoff RPKM \geq 1. (B) Density plot of H3K4me1 (enhancer mark) cutoff RPKM \geq 1. (C) Density plot of H3K27ac (enhancer mark) cutoff RPKM \geq 1. (D) Density plot of short-short paired GRO-seq transcription (SSP) (enhancer mark) cutoff RPKM \geq 1. (E) Density plot of short-unpaired GRO-seq transcription (SUNP) ( enhancer mark) cutoff RPKM \geq 0.5. (F) Density plot of RNA-seq (gene expression) cutoff FPKM \geq 0.4" id="fig:density_plot" data-tag="S2" /><figcaption><span>Figure S2:</span> <strong>Density plots of enhancer and gene expression levels across all cell types.</strong> Kernel density plots of log-transformed RPKM and FPKM values for determining active enhancers and genes. The dashed grey line represents the minimum expression cutoff. <strong>(A)</strong> Density plot of H3K4me3 (promoter mark) cutoff RPKM <span class="math inline">\(\geq 1\)</span>. <strong>(B)</strong> Density plot of H3K4me1 (enhancer mark) cutoff RPKM <span class="math inline">\(\geq 1\)</span>. <strong>(C)</strong> Density plot of H3K27ac (enhancer mark) cutoff RPKM <span class="math inline">\(\geq 1\)</span>. <strong>(D)</strong> Density plot of short-short paired GRO-seq transcription (SSP) (enhancer mark) cutoff RPKM <span class="math inline">\(\geq 1\)</span>. <strong>(E)</strong> Density plot of short-unpaired GRO-seq transcription (SUNP) ( enhancer mark) cutoff RPKM <span class="math inline">\(\geq 0.5\)</span>. <strong>(F)</strong> Density plot of RNA-seq (gene expression) cutoff FPKM <span class="math inline">\(\geq 0.4\)</span></figcaption>
</figure>
<figure>
<img src="images/genome_browser.png" alt="Figure S3: Enhancer transcription is a better predictor of enhancer activity and target gene expression than other features of active chromatin. (A-D) UCSC Genome browser views of GRO-seq, histone modification ChIP-seq and RNA-seq data showing a transcribed enhancer (black box with dashed line) and its nearest neighboring gene. hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). (A) Browser view showing a transcribed enhancer and its nearest neighboring gene (SMAD7). The data highlights histone modifications typically enriched at enhancers (green), however the increased transcription determined by GRO-seq (red/blue) for DE correlates to expression of nearest genes determined by RNA-seq (orange/light green). (B) Browser view showing a transcribed enhancer and its nearest neighboring gene (ATG5. The data highlights an enhancer identified by GRO-seq (red/blue), however lacks typical histone modifications enriched at enhancers (green). The increased transcription determined by GRO-seq for hESC correlates to expression of nearest genes determined by RNA-seq (orange/light green). (C) Browser view showing a transcribed enhancer and its nearest neighboring gene (PDX1). The data highlights an enhancer identified by histone modifications enriched at enhancers (green), however increased transcription determined by GRO-seq (red/blue) correlates with antisense gene (AS-PDX1). (D) Browser view showing a transcribed enhancer and its nearest neighboring gene (RGS4). The data highlights an enhancer identified by histone modifications enriched at enhancers (green), however lacks enhancer transcription identified by GRO-seq (red/blue). The increased enhancer signal determined by histone modifications for PE shows correlates to expression of nearest genes determined by RNA-seq (orange/light green) and GRO-seq (red/blue)." id="fig:genome_browser" data-tag="S3" /><figcaption><span>Figure S3:</span> <strong>Enhancer transcription is a better predictor of enhancer activity and target gene expression than other features of active chromatin.</strong> <strong>(A-D)</strong> UCSC Genome browser views of GRO-seq, histone modification ChIP-seq and RNA-seq data showing a transcribed enhancer <em>(black box with dashed line)</em> and its nearest neighboring gene. hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). <strong>(A)</strong> Browser view showing a transcribed enhancer and its nearest neighboring gene (SMAD7). The data highlights histone modifications typically enriched at enhancers <em>(green)</em>, however the increased transcription determined by GRO-seq <em>(red/blue)</em> for DE correlates to expression of nearest genes determined by RNA-seq <em>(orange/light green)</em>. <strong>(B)</strong> Browser view showing a transcribed enhancer and its nearest neighboring gene (ATG5. The data highlights an enhancer identified by GRO-seq <em>(red/blue)</em>, however lacks typical histone modifications enriched at enhancers <em>(green</em>). The increased transcription determined by GRO-seq for hESC correlates to expression of nearest genes determined by RNA-seq <em>(orange/light green)</em>. <strong>(C)</strong> Browser view showing a transcribed enhancer and its nearest neighboring gene (PDX1). The data highlights an enhancer identified by histone modifications enriched at enhancers <em>(green)</em>, however increased transcription determined by GRO-seq <em>(red/blue)</em> correlates with antisense gene (AS-PDX1). <strong>(D)</strong> Browser view showing a transcribed enhancer and its nearest neighboring gene (RGS4). The data highlights an enhancer identified by histone modifications enriched at enhancers <em>(green)</em>, however lacks enhancer transcription identified by GRO-seq <em>(red/blue)</em>. The increased enhancer signal determined by histone modifications for PE shows correlates to expression of nearest genes determined by RNA-seq <em>(orange/light green)</em> and GRO-seq <em>(red/blue)</em>.</figcaption>
</figure>
<figure>
<img src="images/tfsee_histone.png" alt="Figure S4: TFSEE defined by histone modifications identifies cell type-specific enhancers and their cognate TFs that drive gene expression in pancreatic differentiation. (A) Unsupervised hierarchical clustering of cell line normalized TFSEE scores shown in a heatmap representation. (B) Biaxial t-SNE clustering plot of cell type-normalized TFSEE scores showing evidence of three distinct clusters, each point represents an individual TF. (C) Boxplots of normalized TFSEE score for clusters identified in pancreatic differentiation. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test, p \lt 1 \times 10^{-2}). Number of TFs in each cluster are in parenthesis. Cluster 1, TFs associated across pancreatic lineage Cluster 2, TFs associated with pre-pancreatic lineage induction (hESC, DE and GT). Cluster 3, TFs associated with late-pancreatic differentiation (FG and PE)." id="fig:tfsee_histone" data-tag="S4" /><figcaption><span>Figure S4:</span> <strong>TFSEE defined by histone modifications identifies cell type-specific enhancers and their cognate TFs that drive gene expression in pancreatic differentiation.</strong> <strong>(A)</strong> Unsupervised hierarchical clustering of cell line normalized TFSEE scores shown in a heatmap representation.<br />
<strong>(B)</strong> Biaxial t-SNE clustering plot of cell type-normalized TFSEE scores showing evidence of three distinct clusters, each point represents an individual TF. <strong>(C)</strong> Boxplots of normalized TFSEE score for clusters identified in pancreatic differentiation. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test, <span class="math inline">\(p \lt 1 \times 10^{-2}\)</span>). Number of TFs in each cluster are in parenthesis. Cluster 1, TFs associated across pancreatic lineage Cluster 2, TFs associated with pre-pancreatic lineage induction (hESC, DE and GT). Cluster 3, TFs associated with late-pancreatic differentiation (FG and PE).</figcaption>
</figure>
<figure>
<img src="images/late_pre_histone.png" alt="Figure S5: TFSEE-Predicted TFs, by histone modifications, are enriched in pre- and late- pancreatic differentiation. (A-C) Box plots of normalized TF expression (panel A), enhancer transcription (panel B), and gene expression for the nearest neighboring genes to active enhancers (panel C) in pre- (cluster 2) and late-pancreatic (cluster 3) differentiation across the different cell types. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test). hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). (A) TFs identified in cluster 2 by TFSEE show equal expression across differentiation. While, cluster 3 highlights TFs highly expressed in FG and PE. TF expression as measured by RNA-seq. Number of TFs in each cluster are in parenthesis. (p \lt 1 \times 10^{-4}) (B) Enhancer transcription as measured by ChIP-seq (H3K27ac enrichment). Number of enhancers in each cluster are in parenthesis. (p \lt 1 \times 10^{-4}). (C) Gene expression as measured by RNA-seq. Number of genes in each cluster are in parenthesis. (p \lt 0.05). (D and E) Rank order of TFs enriched in the Cluster 2 and the Cluster 3 identified using TFSEE. The top ten TFs in each Cluster are noted." id="fig:late_pre_histone" data-tag="S5" /><figcaption><span>Figure S5:</span> <strong>TFSEE-Predicted TFs, by histone modifications, are enriched in pre- and late- pancreatic differentiation.</strong> <strong>(A-C)</strong> Box plots of normalized TF expression (panel A), enhancer transcription (panel B), and gene expression for the nearest neighboring genes to active enhancers (panel C) in pre- (cluster 2) and late-pancreatic (cluster 3) differentiation across the different cell types. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test). hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). <strong>(A)</strong> TFs identified in cluster 2 by TFSEE show equal expression across differentiation. While, cluster 3 highlights TFs highly expressed in FG and PE. TF expression as measured by RNA-seq. Number of TFs in each cluster are in parenthesis. (<span class="math inline">\(p \lt 1 \times 10^{-4}\)</span>) <strong>(B)</strong> Enhancer transcription as measured by ChIP-seq (H3K27ac enrichment). Number of enhancers in each cluster are in parenthesis. (<span class="math inline">\(p \lt 1 \times 10^{-4}\)</span>). <strong>(C)</strong> Gene expression as measured by RNA-seq. Number of genes in each cluster are in parenthesis. (<span class="math inline">\(p \lt 0.05\)</span>). <strong>(D and E)</strong> Rank order of TFs enriched in the Cluster 2 and the Cluster 3 identified using TFSEE. The top ten TFs in each Cluster are noted.</figcaption>
</figure>
<figure>
<img src="images/primitive_gut.png" alt="Figure S6: TFSEE-Predicted TFs are enriched and depleted in Primitive Gut Tube during pancreatic differentiation. (A-C) Box plots of normalized TF expression (panel A), enhancer transcription (panel B), and gene expression for the nearest neighboring genes to active enhancers (panel C) in depleted (cluster 1) and enriched (cluster 2) in primitive gut tube during pancreatic differentiation across different cell types. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test). hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). (A) TF expression as measured by RNA-seq. Number of TFs in each cluster are in parenthesis. (p \lt 1 \times 10^{-2}) (B) Enhancer transcription as measured by GRO-seq. Number of enhancers in each cluster are in parenthesis. (p \lt 1 \times 10^{-4}). (C) Gene expression as measured by RNA-seq. Number of genes in each cluster are in parenthesis. (p \lt 0.05). (D and E) Rank order of TFs enriched in the Cluster 1 and the Cluster 2 identified using TFSEE. The top five TFs in each Cluster are noted." id="fig:primitive_gut" data-tag="S6" /><figcaption><span>Figure S6:</span> <strong>TFSEE-Predicted TFs are enriched and depleted in Primitive Gut Tube during pancreatic differentiation.</strong> <strong>(A-C)</strong> Box plots of normalized TF expression (panel A), enhancer transcription (panel B), and gene expression for the nearest neighboring genes to active enhancers (panel C) in depleted (cluster 1) and enriched (cluster 2) in primitive gut tube during pancreatic differentiation across different cell types. Bars marked with different letters are significantly different from each other (Wilcoxon rank sum test). hESC (human embryonic stem cell); DE (definitive endoderm); GT (primitive gut tube); FG (posterior foregut); PE (pancreatic endoderm). <strong>(A)</strong> TF expression as measured by RNA-seq. Number of TFs in each cluster are in parenthesis. (<span class="math inline">\(p \lt 1 \times 10^{-2}\)</span>) <strong>(B)</strong> Enhancer transcription as measured by GRO-seq. Number of enhancers in each cluster are in parenthesis. (<span class="math inline">\(p \lt 1 \times 10^{-4}\)</span>). <strong>(C)</strong> Gene expression as measured by RNA-seq. Number of genes in each cluster are in parenthesis. (<span class="math inline">\(p \lt 0.05\)</span>). <strong>(D and E)</strong> Rank order of TFs enriched in the Cluster 1 and the Cluster 2 identified using TFSEE. The top five TFs in each Cluster are noted.</figcaption>
</figure>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-sJDRq7sP">
<p>1. <strong>Transcriptional enhancers: from properties to genome-wide predictions</strong><br />
Daria Shlyueva, Gerald Stampfel, Alexander Stark<br />
<em>Nature Reviews Genetics</em> (2014-03-11) <a href="https://doi.org/10.1038/nrg3682" class="uri">https://doi.org/10.1038/nrg3682</a></p>
</div>
<div id="ref-d09ELIR1">
<p>2. <strong>The selection and function of cell type-specific enhancers</strong><br />
Sven Heinz, Casey E. Romanoski, Christopher Benner, Christopher K. Glass<br />
<em>Nature Reviews Molecular Cell Biology</em> (2015-02-04) <a href="https://doi.org/10.1038/nrm3949" class="uri">https://doi.org/10.1038/nrm3949</a></p>
</div>
<div id="ref-r7FNQqrP">
<p>3. <strong>Progress and challenges in bioinformatics approaches for enhancer identification</strong><br />
Dimitrios Kleftogiannis, Panos Kalnis, Vladimir B. Bajic<br />
<em>Briefings in Bioinformatics</em> (2015-12-03) <a href="https://doi.org/10.1093/bib/bbv101" class="uri">https://doi.org/10.1093/bib/bbv101</a></p>
</div>
<div id="ref-15J98V2qM">
<p>4. <strong>An integrated encyclopedia of DNA elements in the human genome</strong><em>Nature</em> (2012-09) <a href="https://doi.org/10.1038/nature11247" class="uri">https://doi.org/10.1038/nature11247</a></p>
</div>
<div id="ref-sLkFMFZj">
<p>5. <strong>Integrative analysis of 111 reference human epigenomes</strong><br />
Anshul Kundaje, Wouter Meuleman, Jason Ernst, Misha Bilenky, Angela Yen, Alireza Heravi-Moussavi, Pouya Kheradpour, Zhizhuo Zhang, Jianrong Wang, Michael J. Ziller, … Manolis Kellis<br />
<em>Nature</em> (2015-02-18) <a href="https://doi.org/10.1038/nature14248" class="uri">https://doi.org/10.1038/nature14248</a></p>
</div>
<div id="ref-NqDGzVRS">
<p>6. <strong>Genome-wide mapping of DNase hypersensitive sites using massively parallel signature sequencing (MPSS)</strong><br />
G. E. Crawford<br />
<em>Genome Research</em> (2005-12-12) <a href="https://doi.org/10.1101/gr.4074106" class="uri">https://doi.org/10.1101/gr.4074106</a></p>
</div>
<div id="ref-x2PdIgDj">
<p>7. <strong>Patterns of regulatory activity across diverse human cell types predict tissue identity, transcription factor binding, and long-range interactions</strong><br />
N. C. Sheffield, R. E. Thurman, L. Song, A. Safi, J. A. Stamatoyannopoulos, B. Lenhard, G. E. Crawford, T. S. Furey<br />
<em>Genome Research</em> (2013-03-12) <a href="https://doi.org/10.1101/gr.152140.112" class="uri">https://doi.org/10.1101/gr.152140.112</a></p>
</div>
<div id="ref-6xCXy2Jy">
<p>8. <strong>Discovery of Transcription Factors and Regulatory Regions Driving In Vivo Tumor Development by ATAC-seq and FAIRE-seq Open Chromatin Profiling</strong><br />
Kristofer Davie, Jelle Jacobs, Mardelle Atkins, Delphine Potier, Valerie Christiaens, Georg Halder, Stein Aerts<br />
<em>PLOS Genetics</em> (2015-02-13) <a href="https://doi.org/10.1371/journal.pgen.1004994" class="uri">https://doi.org/10.1371/journal.pgen.1004994</a></p>
</div>
<div id="ref-vCijBy88">
<p>9. <strong>Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome</strong><br />
Nathaniel D Heintzman, Rhona K Stuart, Gary Hon, Yutao Fu, Christina W Ching, R David Hawkins, Leah O Barrera, Sara Van Calcar, Chunxu Qu, Keith A Ching, … Bing Ren<br />
<em>Nature Genetics</em> (2007-02-04) <a href="https://doi.org/10.1038/ng1966" class="uri">https://doi.org/10.1038/ng1966</a></p>
</div>
<div id="ref-O5G8lPNO">
<p>10. <strong>Histone modifications at human enhancers reflect global cell-type-specific gene expression</strong><br />
Nathaniel D. Heintzman, Gary C. Hon, R. David Hawkins, Pouya Kheradpour, Alexander Stark, Lindsey F. Harp, Zhen Ye, Leonard K. Lee, Rhona K. Stuart, Christina W. Ching, … Bing Ren<br />
<em>Nature</em> (2009-03-18) <a href="https://doi.org/10.1038/nature07829" class="uri">https://doi.org/10.1038/nature07829</a></p>
</div>
<div id="ref-LkrR69b9">
<p>11. <strong>Distinct and Predictive Histone Lysine Acetylation Patterns at Promoters, Enhancers, and Gene Bodies</strong><br />
Nisha Rajagopal, Jason Ernst, Pradipta Ray, Jie Wu, Michael Zhang, Manolis Kellis, Bing Ren<br />
<em>G3&amp;#58; Genes|Genomes|Genetics</em> (2014-08-12) <a href="https://doi.org/10.1534/g3.114.013565" class="uri">https://doi.org/10.1534/g3.114.013565</a></p>
</div>
<div id="ref-1CNxOwAG6">
<p>12. <strong>Effects on the transcriptome upon deletion of a distal element cannot be predicted by the size of the H3K27Ac peak in human cells</strong><br />
Yu Gyoung Tak, Yuli Hung, Lijing Yao, Matthew R. Grimmer, Albert Do, Mital S. Bhakta, Henriette O’Geen, David J. Segal, Peggy J. Farnham<br />
<em>Nucleic Acids Research</em> (2016-01-06) <a href="https://doi.org/10.1093/nar/gkv1530" class="uri">https://doi.org/10.1093/nar/gkv1530</a></p>
</div>
<div id="ref-UIPf9Y7q">
<p>13. <strong>Occupancy by key transcription factors is a more accurate predictor of enhancer activity than histone modifications or chromatin accessibility</strong><br />
Nergiz Dogan, Weisheng Wu, Christapher S Morrissey, Kuan-Bei Chen, Aaron Stonestrom, Maria Long, Cheryl A Keller, Yong Cheng, Deepti Jain, Axel Visel, … Ross C Hardison<br />
<em>Epigenetics &amp; Chromatin</em> (2015-04-23) <a href="https://doi.org/10.1186/s13072-015-0009-5" class="uri">https://doi.org/10.1186/s13072-015-0009-5</a></p>
</div>
<div id="ref-3ZN37fOe">
<p>14. <strong>A Large Fraction of Extragenic RNA Pol II Transcription Sites Overlap Enhancers</strong><br />
Francesca De Santa, Iros Barozzi, Flore Mietton, Serena Ghisletti, Sara Polletti, Betsabeh Khoramian Tusi, Heiko Muller, Jiannis Ragoussis, Chia-Lin Wei, Gioacchino Natoli<br />
<em>PLoS Biology</em> (2010-05-11) <a href="https://doi.org/10.1371/journal.pbio.1000384" class="uri">https://doi.org/10.1371/journal.pbio.1000384</a></p>
</div>
<div id="ref-5i6RU9Zx">
<p>15. <strong>Architectural and Functional Commonalities between Enhancers and Promoters</strong><br />
Tae-Kyung Kim, Ramin Shiekhattar<br />
<em>Cell</em> (2015-08) <a href="https://doi.org/10.1016/j.cell.2015.08.008" class="uri">https://doi.org/10.1016/j.cell.2015.08.008</a></p>
</div>
<div id="ref-1BkIYUDLC">
<p>16. <strong>A Rapid, Extensive, and Transient Transcriptional Response to Estrogen Signaling in Breast Cancer Cells</strong><br />
Nasun Hah, Charles G. Danko, Leighton Core, Joshua J. Waterfall, Adam Siepel, John T. Lis, W. Lee Kraus<br />
<em>Cell</em> (2011-05) <a href="https://doi.org/10.1016/j.cell.2011.03.042" class="uri">https://doi.org/10.1016/j.cell.2011.03.042</a></p>
</div>
<div id="ref-9nnd7Ryc">
<p>17. <strong>Tissue-Specific RNA Expression Marks Distant-Acting Developmental Enhancers</strong><br />
Han Wu, Alex S. Nord, Jennifer A. Akiyama, Malak Shoukry, Veena Afzal, Edward M. Rubin, Len A. Pennacchio, Axel Visel<br />
<em>PLoS Genetics</em> (2014-09-04) <a href="https://doi.org/10.1371/journal.pgen.1004610" class="uri">https://doi.org/10.1371/journal.pgen.1004610</a></p>
</div>
<div id="ref-14GXIWfT0">
<p>18. <strong>Reprogramming transcription by distinct classes of enhancers functionally defined by eRNA</strong><br />
Dong Wang, Ivan Garcia-Bassets, Chris Benner, Wenbo Li, Xue Su, Yiming Zhou, Jinsong Qiu, Wen Liu, Minna U. Kaikkonen, Kenneth A. Ohgi, … Xiang-Dong Fu<br />
<em>Nature</em> (2011-05-15) <a href="https://doi.org/10.1038/nature10006" class="uri">https://doi.org/10.1038/nature10006</a></p>
</div>
<div id="ref-1DE6cLccI">
<p>19. <strong>Enhancer transcripts mark active estrogen receptor binding sites</strong><br />
N. Hah, S. Murakami, A. Nagari, C. G. Danko, W. L. Kraus<br />
<em>Genome Research</em> (2013-05-01) <a href="https://doi.org/10.1101/gr.152306.112" class="uri">https://doi.org/10.1101/gr.152306.112</a></p>
</div>
<div id="ref-18olkFjWD">
<p>20. <strong>Analysis of nascent RNA identifies a unified architecture of initiation regions at mammalian promoters and enhancers</strong><br />
Leighton J Core, André L Martins, Charles G Danko, Colin T Waters, Adam Siepel, John T Lis<br />
<em>Nature Genetics</em> (2014-11-10) <a href="https://doi.org/10.1038/ng.3142" class="uri">https://doi.org/10.1038/ng.3142</a></p>
</div>
<div id="ref-lPKPHCBB">
<p>21. <strong>groHMM: a computational tool for identifying unannotated and cell type-specific transcription units from global run-on sequencing data</strong><br />
Minho Chae, Charles G. Danko, W. Lee Kraus<br />
<em>BMC Bioinformatics</em> (2015-07-16) <a href="https://doi.org/10.1186/s12859-015-0656-3" class="uri">https://doi.org/10.1186/s12859-015-0656-3</a></p>
</div>
<div id="ref-GvDoUOfC">
<p>22. <strong>Functional roles of enhancer RNAs for oestrogen-dependent transcriptional activation</strong><br />
Wenbo Li, Dimple Notani, Qi Ma, Bogdan Tanasa, Esperanza Nunez, Aaron Yun Chen, Daria Merkurjev, Jie Zhang, Kenneth Ohgi, Xiaoyuan Song, … Michael G. Rosenfeld<br />
<em>Nature</em> (2013-06) <a href="https://doi.org/10.1038/nature12210" class="uri">https://doi.org/10.1038/nature12210</a></p>
</div>
<div id="ref-2eAh2YIr">
<p>23. <strong>TNFα Signaling Exposes Latent Estrogen Receptor Binding Sites to Alter the Breast Cancer Cell Transcriptome</strong><br />
Hector L. Franco, Anusha Nagari, W. Lee Kraus<br />
<em>Molecular Cell</em> (2015-04) <a href="https://doi.org/10.1016/j.molcel.2015.02.001" class="uri">https://doi.org/10.1016/j.molcel.2015.02.001</a></p>
</div>
<div id="ref-14L6kLUE3">
<p>24. <strong>VISTA Enhancer Browser–a database of tissue-specific human enhancers</strong><br />
A. Visel, S. Minovitsky, I. Dubchak, L. A. Pennacchio<br />
<em>Nucleic Acids Research</em> (2007-01-03) <a href="https://doi.org/10.1093/nar/gkl822" class="uri">https://doi.org/10.1093/nar/gkl822</a></p>
</div>
<div id="ref-sKU362wY">
<p>25. <strong>Systematic dissection of regulatory motifs in 2000 predicted human enhancers using a massively parallel reporter assay</strong><br />
P. Kheradpour, J. Ernst, A. Melnikov, P. Rogov, L. Wang, X. Zhang, J. Alston, T. S. Mikkelsen, M. Kellis<br />
<em>Genome Research</em> (2013-03-19) <a href="https://doi.org/10.1101/gr.144899.112" class="uri">https://doi.org/10.1101/gr.144899.112</a></p>
</div>
<div id="ref-jjAPfyKA">
<p>26. <strong>Genome-Wide Quantitative Enhancer Activity Maps Identified by STARR-seq</strong><br />
C. D. Arnold, D. Gerlach, C. Stelzer, L. M. Boryn, M. Rath, A. Stark<br />
<em>Science</em> (2013-01-17) <a href="https://doi.org/10.1126/science.1232542" class="uri">https://doi.org/10.1126/science.1232542</a></p>
</div>
<div id="ref-5zmE7Qkb">
<p>27. <strong>High-throughput functional testing of ENCODE segmentation predictions</strong><br />
Jamie C. Kwasnieski, Christopher Fiore, Hemangi G. Chaudhari, Barak A. Cohen<br />
<em>Genome Research</em> (2014-07-17) <a href="https://doi.org/10.1101/gr.173518.114" class="uri">https://doi.org/10.1101/gr.173518.114</a></p>
</div>
<div id="ref-ZnmOeYfS">
<p>28. <strong>Architecture of the human regulatory network derived from ENCODE data</strong><br />
Mark B. Gerstein, Anshul Kundaje, Manoj Hariharan, Stephen G. Landt, Koon-Kiu Yan, Chao Cheng, Xinmeng Jasmine Mu, Ekta Khurana, Joel Rozowsky, Roger Alexander, … Michael Snyder<br />
<em>Nature</em> (2012-09) <a href="https://doi.org/10.1038/nature11245" class="uri">https://doi.org/10.1038/nature11245</a></p>
</div>
<div id="ref-EkABaVh5">
<p>29. <strong>A census of human transcription factors: function, expression and evolution</strong><br />
Juan M. Vaquerizas, Sarah K. Kummerfeld, Sarah A. Teichmann, Nicholas M. Luscombe<br />
<em>Nature Reviews Genetics</em> (2009-04) <a href="https://doi.org/10.1038/nrg2538" class="uri">https://doi.org/10.1038/nrg2538</a></p>
</div>
<div id="ref-uOXPLhWz">
<p>30. <strong>DNA-dependent formation of transcription factor pairs alters their binding specificity</strong><br />
Arttu Jolma, Yimeng Yin, Kazuhiro R. Nitta, Kashyap Dave, Alexander Popov, Minna Taipale, Martin Enge, Teemu Kivioja, Ekaterina Morgunova, Jussi Taipale<br />
<em>Nature</em> (2015-11-09) <a href="https://doi.org/10.1038/nature15518" class="uri">https://doi.org/10.1038/nature15518</a></p>
</div>
<div id="ref-1AqKyPkB5">
<p>31. <strong>DNA-Binding Specificities of Human Transcription Factors</strong><br />
Arttu Jolma, Jian Yan, Thomas Whitington, Jarkko Toivonen, Kazuhiro R. Nitta, Pasi Rastas, Ekaterina Morgunova, Martin Enge, Mikko Taipale, Gonghong Wei, … Jussi Taipale<br />
<em>Cell</em> (2013-01) <a href="https://doi.org/10.1016/j.cell.2012.12.009" class="uri">https://doi.org/10.1016/j.cell.2012.12.009</a></p>
</div>
<div id="ref-1DrZNoXOU">
<p>32. <strong>JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles</strong><br />
Anthony Mathelier, Oriol Fornes, David J. Arenillas, Chih-yu Chen, Grégoire Denay, Jessica Lee, Wenqiang Shi, Casper Shyr, Ge Tan, Rebecca Worsley-Hunt, … Wyeth W. Wasserman<br />
<em>Nucleic Acids Research</em> (2015-11-03) <a href="https://doi.org/10.1093/nar/gkv1176" class="uri">https://doi.org/10.1093/nar/gkv1176</a></p>
</div>
<div id="ref-1C9AIanMe">
<p>33. <strong>Quantifying similarity between motifs</strong><br />
Shobhit Gupta, John A Stamatoyannopoulos, Timothy L Bailey, William Noble<br />
<em>Genome Biology</em> (2007) <a href="https://doi.org/10.1186/gb-2007-8-2-r24" class="uri">https://doi.org/10.1186/gb-2007-8-2-r24</a></p>
</div>
<div id="ref-uuBDz4a3">
<p>34. <strong>Functional analysis of transcription factor binding sites in human promoters</strong><br />
Troy W Whitfield, Jie Wang, Patrick J Collins, E Christopher Partridge, Shelley Aldred, Nathan D Trinklein, Richard M Myers, Zhiping Weng<br />
<em>Genome Biology</em> (2012) <a href="https://doi.org/10.1186/gb-2012-13-9-r50" class="uri">https://doi.org/10.1186/gb-2012-13-9-r50</a></p>
</div>
<div id="ref-Iu6Tq2rd">
<p>35. <strong>Enhancer transcription reveals subtype-specific gene expression programs controlling breast cancer pathogenesis</strong><br />
Hector L. Franco, Anusha Nagari, Venkat S. Malladi, Wenqian Li, Yuanxin Xi, Dana Richardson, Kendra L. Allton, Kaori Tanaka, Jing Li, Shino Murakami, … W. Lee Kraus<br />
<em>Genome Research</em> (2017-12-22) <a href="https://doi.org/10.1101/gr.226019.117" class="uri">https://doi.org/10.1101/gr.226019.117</a></p>
</div>
<div id="ref-BTqz3DNL">
<p>36. <strong>Dynamic Chromatin Remodeling Mediated by Polycomb Proteins Orchestrates Pancreatic Differentiation of Human Embryonic Stem Cells</strong><br />
Ruiyu Xie, Logan J. Everett, Hee-Woong Lim, Nisha A. Patel, Jonathan Schug, Evert Kroon, Olivia G. Kelly, Allen Wang, Kevin A. D’Amour, Allan J. Robins, … Maike Sander<br />
<em>Cell Stem Cell</em> (2013-02) <a href="https://doi.org/10.1016/j.stem.2012.11.023" class="uri">https://doi.org/10.1016/j.stem.2012.11.023</a></p>
</div>
<div id="ref-18a6G1TE5">
<p>37. <strong>Epigenetic Priming of Enhancers Predicts Developmental Competence of hESC-Derived Endodermal Lineage Intermediates</strong><br />
Allen Wang, Feng Yue, Yan Li, Ruiyu Xie, Thomas Harper, Nisha A. Patel, Kayla Muth, Jeffrey Palmer, Yunjiang Qiu, Jinzhao Wang, … Maike Sander<br />
<em>Cell Stem Cell</em> (2015-04) <a href="https://doi.org/10.1016/j.stem.2015.02.013" class="uri">https://doi.org/10.1016/j.stem.2015.02.013</a></p>
</div>
<div id="ref-15rkMH6SD">
<p>38. <strong>GENCODE: The reference human genome annotation for The ENCODE Project</strong><br />
J. Harrow, A. Frankish, J. M. Gonzalez, E. Tapanari, M. Diekhans, F. Kokocinski, B. L. Aken, D. Barrell, A. Zadissa, S. Searle, … T. J. Hubbard<br />
<em>Genome Research</em> (2012-09-01) <a href="https://doi.org/10.1101/gr.135350.111" class="uri">https://doi.org/10.1101/gr.135350.111</a></p>
</div>
<div id="ref-9O9ugXze">
<p>39. <strong>A Chromatin Landmark and Transcription Initiation at Most Promoters in Human Cells</strong><br />
Matthew G. Guenther, Stuart S. Levine, Laurie A. Boyer, Rudolf Jaenisch, Richard A. Young<br />
<em>Cell</em> (2007-07) <a href="https://doi.org/10.1016/j.cell.2007.05.042" class="uri">https://doi.org/10.1016/j.cell.2007.05.042</a></p>
</div>
<div id="ref-crCcep4D">
<p>40. <strong>Predicting enhancer transcription and activity from chromatin modifications</strong><br />
Yun Zhu, Lin Sun, Zhao Chen, John W. Whitaker, Tao Wang, Wei Wang<br />
<em>Nucleic Acids Research</em> (2013-09-12) <a href="https://doi.org/10.1093/nar/gkt826" class="uri">https://doi.org/10.1093/nar/gkt826</a></p>
</div>
<div id="ref-zQrKrStO">
<p>41. <strong>Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture–on-chip (4C)</strong><br />
Marieke Simonis, Petra Klous, Erik Splinter, Yuri Moshkin, Rob Willemsen, Elzo de Wit, Bas van Steensel, Wouter de Laat<br />
<em>Nature Genetics</em> (2006-10-08) <a href="https://doi.org/10.1038/ng1896" class="uri">https://doi.org/10.1038/ng1896</a></p>
</div>
<div id="ref-Z4cumk9R">
<p>42. <strong>ChIA-PET analysis of transcriptional chromatin interactions</strong><br />
Jingyao Zhang, Huay Mei Poh, Su Qin Peh, Yee Yen Sia, Guoliang Li, Fabianus Hendriyan Mulawadi, Yufen Goh, Melissa J. Fullwood, Wing-Kin Sung, Xiaoan Ruan, Yijun Ruan<br />
<em>Methods</em> (2012-11) <a href="https://doi.org/10.1016/j.ymeth.2012.08.009" class="uri">https://doi.org/10.1016/j.ymeth.2012.08.009</a></p>
</div>
<div id="ref-SNLehHrw">
<p>43. <strong>Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C</strong><br />
Borbala Mifsud, Filipe Tavares-Cadete, Alice N Young, Robert Sugar, Stefan Schoenfelder, Lauren Ferreira, Steven W Wingett, Simon Andrews, William Grey, Philip A Ewels, … Cameron S Osborne<br />
<em>Nature Genetics</em> (2015-05-04) <a href="https://doi.org/10.1038/ng.3286" class="uri">https://doi.org/10.1038/ng.3286</a></p>
</div>
<div id="ref-gM7zvZej">
<p>44. <strong>The histone gene activator HINFP is a nonredundant cyclin E/CDK2 effector during early embryonic cell cycles</strong><br />
R. Xie, R. Medina, Y. Zhang, S. Hussain, J. Colby, P. Ghule, S. Sundararajan, M. Keeler, L.-J. Liu, M. van der Deen, … G. S. Stein<br />
<em>Proceedings of the National Academy of Sciences</em> (2009-07-09) <a href="https://doi.org/10.1073/pnas.0905651106" class="uri">https://doi.org/10.1073/pnas.0905651106</a></p>
</div>
<div id="ref-rj3cD1pL">
<p>45. <strong>Nuclear receptor regulation of stemness and stem cell differentiation</strong><br />
Yangsik Jeong, David J. Mangelsdorf<br />
<em>Experimental and Molecular Medicine</em> (2009) <a href="https://doi.org/10.3858/emm.2009.41.8.091" class="uri">https://doi.org/10.3858/emm.2009.41.8.091</a></p>
</div>
<div id="ref-197694eEU">
<p>46. <strong>Zic3 Is Required for Maintenance of Pluripotency in Embryonic Stem Cells</strong><br />
L. S. Lim, Y.-H. Loh, W. Zhang, Y. Li, X. Chen, Y. Wang, M. Bakre, H.-H. Ng, L. W. Stanton<br />
<em>Molecular Biology of the Cell</em> (2007-01-31) <a href="https://doi.org/10.1091/mbc.e06-07-0624" class="uri">https://doi.org/10.1091/mbc.e06-07-0624</a></p>
</div>
<div id="ref-1EnCQmxEg">
<p>47. <strong>Sp1-like transcription factors are regulators of embryonic development in vertebrates</strong><br />
Chengtian Zhao, Anming Meng<br />
<em>Development, Growth and Differentiation</em> (2005-05) <a href="https://doi.org/10.1111/j.1440-169x.2005.00797.x" class="uri">https://doi.org/10.1111/j.1440-169x.2005.00797.x</a></p>
</div>
<div id="ref-1A0K4olzK">
<p>48. <strong>Role of the Onecut transcription factors in pancreas morphogenesis and in pancreatic and enteric endocrine differentiation</strong><br />
Vinciane Vanhorenbeeck, Marjorie Jenny, Jean-François Cornut, Gérard Gradwohl, Frédéric P. Lemaigre, Guy G. Rousseau, Patrick Jacquemin<br />
<em>Developmental Biology</em> (2007-05) <a href="https://doi.org/10.1016/j.ydbio.2007.02.027" class="uri">https://doi.org/10.1016/j.ydbio.2007.02.027</a></p>
</div>
<div id="ref-8w17x8DY">
<p>49. <strong>Critical Role of Egr Transcription Factors in Regulating Insulin Biosynthesis, Blood Glucose Homeostasis, and Islet Size</strong><br />
Isabelle Müller, Oliver G. Rössler, Christine Wittig, Michael D. Menger, Gerald Thiel<br />
<em>Endocrinology</em> (2012-07) <a href="https://doi.org/10.1210/en.2012-1064" class="uri">https://doi.org/10.1210/en.2012-1064</a></p>
</div>
<div id="ref-1D7e4QOVB">
<p>50. <strong>Microphthalmia Transcription Factor Regulates Pancreatic β-Cell Function</strong><br />
Magdalena A. Mazur, Marcus Winkler, Elvira Ganić, Jesper K. Colberg, Jenny K. Johansson, Hedvig Bennet, Malin Fex, Ulrike A. Nuber, Isabella Artner<br />
<em>Diabetes</em> (2013-04-22) <a href="https://doi.org/10.2337/db12-1464" class="uri">https://doi.org/10.2337/db12-1464</a></p>
</div>
<div id="ref-2TOwbbq7">
<p>51. <strong>The FOXP1, FOXP2 and FOXP4 transcription factors are required for islet alpha cell proliferation and function in mice</strong><br />
Jason M. Spaeth, Chad S. Hunter, Lauren Bonatakis, Min Guo, Catherine A. French, Ian Slack, Manami Hara, Simon E. Fisher, Jorge Ferrer, Edward E. Morrisey, … Roland Stein<br />
<em>Diabetologia</em> (2015-05-29) <a href="https://doi.org/10.1007/s00125-015-3635-3" class="uri">https://doi.org/10.1007/s00125-015-3635-3</a></p>
</div>
<div id="ref-Bc3QGVe7">
<p>52. <strong>GEO</strong><br />
Gene Expression Omnibus<br />
<a href="https://www.ncbi.nlm.nih.gov/geo/" class="uri">https://www.ncbi.nlm.nih.gov/geo/</a></p>
</div>
<div id="ref-937jwJMM">
<p>53. <strong>ArrayExpress</strong><br />
ArrayExpress – functional genomics data<br />
<a href="http://www.ebi.ac.uk/arrayexpress/" class="uri">http://www.ebi.ac.uk/arrayexpress/</a></p>
</div>
<div id="ref-RU8ikjzP">
<p>54. <strong>Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</strong><br />
Ben Langmead, Cole Trapnell, Mihai Pop, Steven L Salzberg<br />
<em>Genome Biology</em> (2009) <a href="https://doi.org/10.1186/gb-2009-10-3-r25" class="uri">https://doi.org/10.1186/gb-2009-10-3-r25</a></p>
</div>
<div id="ref-IPc8cy7s">
<p>55. <strong>The Sequence Alignment/Map format and SAMtools</strong><br />
H. Li, B. Handsaker, A. Wysoker, T. Fennell, J. Ruan, N. Homer, G. Marth, G. Abecasis, R. Durbin, <br />
<em>Bioinformatics</em> (2009-06-08) <a href="https://doi.org/10.1093/bioinformatics/btp352" class="uri">https://doi.org/10.1093/bioinformatics/btp352</a></p>
</div>
<div id="ref-tfF98ztu">
<p>56. <strong>Picard</strong><br />
Broad Institute<br />
<em>GitHub</em> <a href="http://broadinstitute.github.io/picard/" class="uri">http://broadinstitute.github.io/picard/</a></p>
</div>
<div id="ref-1HWiAHnIw">
<p>57. <strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong><br />
Aaron R. Quinlan, Ira M. Hall<br />
<em>Bioinformatics</em> (2010-01-28) <a href="https://doi.org/10.1093/bioinformatics/btq033" class="uri">https://doi.org/10.1093/bioinformatics/btq033</a></p>
</div>
<div id="ref-hvhWxir9">
<p>58. <strong>ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia</strong><br />
S. G. Landt, G. K. Marinov, A. Kundaje, P. Kheradpour, F. Pauli, S. Batzoglou, B. E. Bernstein, P. Bickel, J. B. Brown, P. Cayting, … M. Snyder<br />
<em>Genome Research</em> (2012-09-01) <a href="https://doi.org/10.1101/gr.136184.111" class="uri">https://doi.org/10.1101/gr.136184.111</a></p>
</div>
<div id="ref-Bhecn2fS">
<p>59. <strong>Identifying ChIP-seq enrichment using MACS</strong><br />
Jianxing Feng, Tao Liu, Bo Qin, Yong Zhang, Xiaole Shirley Liu<br />
<em>Nature Protocols</em> (2012-08-30) <a href="https://doi.org/10.1038/nprot.2012.101" class="uri">https://doi.org/10.1038/nprot.2012.101</a></p>
</div>
<div id="ref-tTu8Ds9Z">
<p>60. <strong>STAR: ultrafast universal RNA-seq aligner</strong><br />
Alexander Dobin, Carrie A. Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, Thomas R. Gingeras<br />
<em>Bioinformatics</em> (2012-10-25) <a href="https://doi.org/10.1093/bioinformatics/bts635" class="uri">https://doi.org/10.1093/bioinformatics/bts635</a></p>
</div>
<div id="ref-Dh2n1EV3">
<p>61. <strong>RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</strong><br />
Bo Li, Colin N Dewey<br />
<em>BMC Bioinformatics</em> (2011) <a href="https://doi.org/10.1186/1471-2105-12-323" class="uri">https://doi.org/10.1186/1471-2105-12-323</a></p>
</div>
<div id="ref-jwUQ55cX">
<p>62. <strong>FASTX-Toolkit</strong><br />
Hannon Lab<br />
<a href="http://hannonlab.cshl.edu/fastx_toolkit/" class="uri">http://hannonlab.cshl.edu/fastx_toolkit/</a></p>
</div>
<div id="ref-roi5bwXL">
<p>63. <strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong><br />
H. Li, R. Durbin<br />
<em>Bioinformatics</em> (2009-05-18) <a href="https://doi.org/10.1093/bioinformatics/btp324" class="uri">https://doi.org/10.1093/bioinformatics/btp324</a></p>
</div>
<div id="ref-qxfTM222">
<p>64. <strong>Seaborn: V0.7.1 (June 2016)</strong><br />
Michael Waskom, Olga Botvinnik, Drewokane, Paul Hobson, David, Yaroslav Halchenko, Saulius Lukauskas, John B. Cole, Jordi Warmenhoven, Julian De Ruiter, … Antony Lee<br />
<em>Zenodo</em> (2016-06-05) <a href="https://doi.org/10.5281/zenodo.54844" class="uri">https://doi.org/10.5281/zenodo.54844</a></p>
</div>
<div id="ref-tDy18Yol">
<p>65. <strong>MakeGenecodeTSS</strong><br />
Sarah Djebali<br />
<em>GitHub</em> <a href="https://github.com/sdjebali/MakeGencodeTSS" class="uri">https://github.com/sdjebali/MakeGencodeTSS</a></p>
</div>
<div id="ref-dwj6qSn3">
<p>66. <strong>MEME SUITE: tools for motif discovery and searching</strong><br />
T. L. Bailey, M. Boden, F. A. Buske, M. Frith, C. E. Grant, L. Clementi, J. Ren, W. W. Li, W. S. Noble<br />
<em>Nucleic Acids Research</em> (2009-05-20) <a href="https://doi.org/10.1093/nar/gkp335" class="uri">https://doi.org/10.1093/nar/gkp335</a></p>
</div>
<div id="ref-TfukIx2u">
<p>67. <strong>MEME-ChIP: motif analysis of large DNA datasets</strong><br />
Philip Machanick, Timothy L. Bailey<br />
<em>Bioinformatics</em> (2011-04-12) <a href="https://doi.org/10.1093/bioinformatics/btr189" class="uri">https://doi.org/10.1093/bioinformatics/btr189</a></p>
</div>
<div id="ref-1HVy2rbI">
<p>68. <strong>groHMM</strong><br />
Minho Chae Charles G. Danko<br />
<em>Bioconductor</em> (2017) <a href="https://doi.org/10.18129/b9.bioc.grohmm" class="uri">https://doi.org/10.18129/b9.bioc.grohmm</a></p>
</div>
<div id="ref-N7ngUHQX">
<p>69. <strong>Visualizing data using t-SNE</strong><br />
Laurens van der Maaten, Geoffrey Hinton<br />
<em>Journal of Machine Learning Research 9</em> (2008-11)</p>
</div>
<div id="ref-vHIKEjNs">
<p>70. <strong>Scikit-learn: Machine Learning in Python</strong><br />
Fabian Pedregosa, Gaël Varoquaux, Alexandre Gramfort, Vincent Michel, Bertrand Thirion, Olivier Grisel, Mathieu Blondel, Gilles Louppe, Peter Prettenhofer, Ron Weiss, … Édouard Duchesnay<br />
<em>arXiv</em> (2012-01-02) <a href="https://arxiv.org/abs/1201.0490v3" class="uri">https://arxiv.org/abs/1201.0490v3</a></p>
</div>
<div id="ref-GGWEfSeb">
<p>71. <strong>Visualizing Large-scale and High-dimensional Data</strong><br />
Jian Tang, Jingzhou Liu, Ming Zhang, Qiaozhu Mei<br />
<em>Proceedings of the 25th International Conference on World Wide Web - WWW ’16</em> (2016) <a href="https://doi.org/10.1145/2872427.2883041" class="uri">https://doi.org/10.1145/2872427.2883041</a></p>
</div>
<div id="ref-r8BJ7tJV">
<p>72. <strong>Scikit-Learn: 0.17.1 Release Tag For Doi</strong><br />
Olivier Grisel, Andreas Mueller, Fabian Pedregosa, Lars, Alexandre Gramfort, Gilles Louppe, Peter Prettenhofer, Mathieu Blondel, Vlad Niculae, Arnaud Joly, … Maheshakya Wijewardena<br />
<em>Zenodo</em> (2016-04-17) <a href="https://doi.org/10.5281/zenodo.49911" class="uri">https://doi.org/10.5281/zenodo.49911</a></p>
</div>
<div id="ref-1026Gxdsi">
<p>73. <strong>Matplotlib: A 2D Graphics Environment</strong><br />
John D. Hunter<br />
<em>Computing in Science &amp; Engineering</em> (2007) <a href="https://doi.org/10.1109/mcse.2007.55" class="uri">https://doi.org/10.1109/mcse.2007.55</a></p>
</div>
<div id="ref-oaruRtzO">
<p>74. <strong>Matplotlib/Matplotlib V2.0.2</strong><br />
Michael Droettboom, Thomas A Caswell, John Hunter, Eric Firing, Jens Hedegaard Nielsen, Nelle Varoquaux, Benjamin Root, Phil Elson, Darren Dale, Jae-Joon Lee, … Nikita Kniazev<br />
<em>Zenodo</em> (2017-05-10) <a href="https://doi.org/10.5281/zenodo.573577" class="uri">https://doi.org/10.5281/zenodo.573577</a></p>
</div>
</div>
<script>
// AnchorJS minified version below.
// Source https://github.com/bryanbraun/anchorjs/blob/064abdd0987f305933ec4982af6d0c1cf2fd0814/anchor.js

/**
 * AnchorJS - v4.0.0 - 2017-06-02
 * https://github.com/bryanbraun/anchorjs
 * Copyright (c) 2017 Bryan Braun; Licensed MIT
 */
!function(A,e){"use strict";"function"==typeof define&&define.amd?define([],e):"object"==typeof module&&module.exports?module.exports=e():(A.AnchorJS=e(),A.anchors=new A.AnchorJS)}(this,function(){"use strict";function A(A){function e(A){A.icon=A.hasOwnProperty("icon")?A.icon:"",A.visible=A.hasOwnProperty("visible")?A.visible:"hover",A.placement=A.hasOwnProperty("placement")?A.placement:"right",A.class=A.hasOwnProperty("class")?A.class:"",A.truncate=A.hasOwnProperty("truncate")?Math.floor(A.truncate):64}function t(A){var e;if("string"==typeof A||A instanceof String)e=[].slice.call(document.querySelectorAll(A));else{if(!(Array.isArray(A)||A instanceof NodeList))throw new Error("The selector provided to AnchorJS was invalid.");e=[].slice.call(A)}return e}function n(){if(null===document.head.querySelector("style.anchorjs")){var A,e=document.createElement("style");e.className="anchorjs",e.appendChild(document.createTextNode("")),void 0===(A=document.head.querySelector('[rel="stylesheet"], style'))?document.head.appendChild(e):document.head.insertBefore(e,A),e.sheet.insertRule(" .anchorjs-link {   opacity: 0;   text-decoration: none;   -webkit-font-smoothing: antialiased;   -moz-osx-font-smoothing: grayscale; }",e.sheet.cssRules.length),e.sheet.insertRule(" *:hover > .anchorjs-link, .anchorjs-link:focus  {   opacity: 1; }",e.sheet.cssRules.length),e.sheet.insertRule(" [data-anchorjs-icon]::after {   content: attr(data-anchorjs-icon); }",e.sheet.cssRules.length),e.sheet.insertRule(' @font-face {   font-family: "anchorjs-icons";   src: url(data:n/a;base64,AAEAAAALAIAAAwAwT1MvMg8yG2cAAAE4AAAAYGNtYXDp3gC3AAABpAAAAExnYXNwAAAAEAAAA9wAAAAIZ2x5ZlQCcfwAAAH4AAABCGhlYWQHFvHyAAAAvAAAADZoaGVhBnACFwAAAPQAAAAkaG10eASAADEAAAGYAAAADGxvY2EACACEAAAB8AAAAAhtYXhwAAYAVwAAARgAAAAgbmFtZQGOH9cAAAMAAAAAunBvc3QAAwAAAAADvAAAACAAAQAAAAEAAHzE2p9fDzz1AAkEAAAAAADRecUWAAAAANQA6R8AAAAAAoACwAAAAAgAAgAAAAAAAAABAAADwP/AAAACgAAA/9MCrQABAAAAAAAAAAAAAAAAAAAAAwABAAAAAwBVAAIAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAMCQAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAg//0DwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAAIAAAACgAAxAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADAAAAAIAAgAAgAAACDpy//9//8AAAAg6cv//f///+EWNwADAAEAAAAAAAAAAAAAAAAACACEAAEAAAAAAAAAAAAAAAAxAAACAAQARAKAAsAAKwBUAAABIiYnJjQ3NzY2MzIWFxYUBwcGIicmNDc3NjQnJiYjIgYHBwYUFxYUBwYGIwciJicmNDc3NjIXFhQHBwYUFxYWMzI2Nzc2NCcmNDc2MhcWFAcHBgYjARQGDAUtLXoWOR8fORYtLTgKGwoKCjgaGg0gEhIgDXoaGgkJBQwHdR85Fi0tOAobCgoKOBoaDSASEiANehoaCQkKGwotLXoWOR8BMwUFLYEuehYXFxYugC44CQkKGwo4GkoaDQ0NDXoaShoKGwoFBe8XFi6ALjgJCQobCjgaShoNDQ0NehpKGgobCgoKLYEuehYXAAAADACWAAEAAAAAAAEACAAAAAEAAAAAAAIAAwAIAAEAAAAAAAMACAAAAAEAAAAAAAQACAAAAAEAAAAAAAUAAQALAAEAAAAAAAYACAAAAAMAAQQJAAEAEAAMAAMAAQQJAAIABgAcAAMAAQQJAAMAEAAMAAMAAQQJAAQAEAAMAAMAAQQJAAUAAgAiAAMAAQQJAAYAEAAMYW5jaG9yanM0MDBAAGEAbgBjAGgAbwByAGoAcwA0ADAAMABAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAP) format("truetype"); }',e.sheet.cssRules.length)}}this.options=A||{},this.elements=[],e(this.options),this.isTouchDevice=function(){return!!("ontouchstart"in window||window.DocumentTouch&&document instanceof DocumentTouch)},this.add=function(A){var i,o,s,c,r,a,h,l,u,d,f,g,p=[];if(e(this.options),"touch"===(g=this.options.visible)&&(g=this.isTouchDevice()?"always":"hover"),A||(A="h2, h3, h4, h5, h6"),0===(i=t(A)).length)return this;for(n(),o=document.querySelectorAll("[id]"),s=[].map.call(o,function(A){return A.id}),r=0;r<i.length;r++)if(this.hasAnchorJSLink(i[r]))p.push(r);else{if(i[r].hasAttribute("id"))c=i[r].getAttribute("id");else if(i[r].hasAttribute("data-anchor-id"))c=i[r].getAttribute("data-anchor-id");else{u=l=this.urlify(i[r].textContent),h=0;do{void 0!==a&&(u=l+"-"+h),a=s.indexOf(u),h+=1}while(-1!==a);a=void 0,s.push(u),i[r].setAttribute("id",u),c=u}d=c.replace(/-/g," "),(f=document.createElement("a")).className="anchorjs-link "+this.options.class,f.href="#"+c,f.setAttribute("aria-label","Anchor link for: "+d),f.setAttribute("data-anchorjs-icon",this.options.icon),"always"===g&&(f.style.opacity="1"),""===this.options.icon&&(f.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(f.style.lineHeight="inherit")),"left"===this.options.placement?(f.style.position="absolute",f.style.marginLeft="-1em",f.style.paddingRight="0.5em",i[r].insertBefore(f,i[r].firstChild)):(f.style.paddingLeft="0.375em",i[r].appendChild(f))}for(r=0;r<p.length;r++)i.splice(p[r]-r,1);return this.elements=this.elements.concat(i),this},this.remove=function(A){for(var e,n,i=t(A),o=0;o<i.length;o++)(n=i[o].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[o]))&&this.elements.splice(e,1),i[o].removeChild(n));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var t=/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\]/g;return this.options.truncate||e(this.options),A.trim().replace(/\'/gi,"").replace(t,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&(" "+A.firstChild.className+" ").indexOf(" anchorjs-link ")>-1,t=A.lastChild&&(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ")>-1;return e||t||!1}}return A});

// Enable links for selected headers
var anchors = new AnchorJS();
anchors.add("h2, h3, h4")
</script>
<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
<script src="https://hypothes.is/embed.js" async></script>
</body>
</html>
